2020-04-19 02:15:31,971 p=13265 u=akokov |   [WARNING]: provided hosts list is empty, only localhost is available. Note that the implicit localhost does not match 'all'

2020-04-19 02:15:31,981 p=13265 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/docker.py) as it seems to be invalid: No
module named 'distutils.spawn'

2020-04-19 02:15:32,014 p=13265 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/iocage.py) as it seems to be invalid: No
module named 'distutils.spawn'

2020-04-19 02:15:32,014 p=13265 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/jail.py) as it seems to be invalid: No
module named 'distutils.spawn'

2020-04-19 02:15:32,014 p=13265 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/kubectl.py) as it seems to be invalid: No
module named 'distutils.spawn'

2020-04-19 02:15:32,015 p=13265 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/libvirt_lxc.py) as it seems to be invalid:
No module named 'distutils.spawn'

2020-04-19 02:15:32,016 p=13265 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/lxd.py) as it seems to be invalid: No
module named 'distutils.spawn'

2020-04-19 02:15:32,022 p=13265 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/oc.py) as it seems to be invalid: No
module named 'distutils.spawn'

2020-04-19 02:15:32,045 p=13265 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/zone.py) as it seems to be invalid: No
module named 'distutils.spawn'

2020-04-19 02:15:32,159 p=13265 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/callback/osx_say.py) as it seems to be invalid: No
module named 'distutils.spawn'

2020-04-19 02:15:32,160 p=13265 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/callback/say.py) as it seems to be invalid: No module
named 'distutils.spawn'

2020-04-19 02:15:32,187 p=13265 u=akokov |   [WARNING]: Could not match supplied host pattern, ignoring: hosts_test

2020-04-19 02:15:32,189 p=13265 u=akokov |  PLAY [hosts_test] ****************************************************************************************************************
2020-04-19 02:15:32,189 p=13265 u=akokov |  skipping: no hosts matched
2020-04-19 02:15:32,189 p=13265 u=akokov |  PLAY RECAP ***********************************************************************************************************************
2020-04-19 02:16:14,421 p=13317 u=akokov |   [WARNING]: provided hosts list is empty, only localhost is available. Note that the implicit localhost does not match 'all'

2020-04-19 02:16:14,431 p=13317 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/docker.py) as it seems to be invalid: No
module named 'distutils.spawn'

2020-04-19 02:16:14,464 p=13317 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/iocage.py) as it seems to be invalid: No
module named 'distutils.spawn'

2020-04-19 02:16:14,464 p=13317 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/jail.py) as it seems to be invalid: No
module named 'distutils.spawn'

2020-04-19 02:16:14,464 p=13317 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/kubectl.py) as it seems to be invalid: No
module named 'distutils.spawn'

2020-04-19 02:16:14,465 p=13317 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/libvirt_lxc.py) as it seems to be invalid:
No module named 'distutils.spawn'

2020-04-19 02:16:14,466 p=13317 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/lxd.py) as it seems to be invalid: No
module named 'distutils.spawn'

2020-04-19 02:16:14,472 p=13317 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/oc.py) as it seems to be invalid: No
module named 'distutils.spawn'

2020-04-19 02:16:14,496 p=13317 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/zone.py) as it seems to be invalid: No
module named 'distutils.spawn'

2020-04-19 02:16:14,612 p=13317 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/callback/osx_say.py) as it seems to be invalid: No
module named 'distutils.spawn'

2020-04-19 02:16:14,613 p=13317 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/callback/say.py) as it seems to be invalid: No module
named 'distutils.spawn'

2020-04-19 02:16:14,639 p=13317 u=akokov |   [WARNING]: Could not match supplied host pattern, ignoring: hosts_test

2020-04-19 02:16:14,641 p=13317 u=akokov |  PLAY [hosts_test] ****************************************************************************************************************
2020-04-19 02:16:14,641 p=13317 u=akokov |  skipping: no hosts matched
2020-04-19 02:16:14,641 p=13317 u=akokov |  PLAY RECAP ***********************************************************************************************************************
2020-04-19 02:16:44,752 p=13395 u=akokov |   [WARNING]: provided hosts list is empty, only localhost is available. Note that the implicit localhost does not match 'all'

2020-04-19 02:16:44,761 p=13395 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/docker.py) as it seems to be invalid: No
module named 'distutils.spawn'

2020-04-19 02:16:44,794 p=13395 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/iocage.py) as it seems to be invalid: No
module named 'distutils.spawn'

2020-04-19 02:16:44,794 p=13395 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/jail.py) as it seems to be invalid: No
module named 'distutils.spawn'

2020-04-19 02:16:44,795 p=13395 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/kubectl.py) as it seems to be invalid: No
module named 'distutils.spawn'

2020-04-19 02:16:44,795 p=13395 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/libvirt_lxc.py) as it seems to be invalid:
No module named 'distutils.spawn'

2020-04-19 02:16:44,797 p=13395 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/lxd.py) as it seems to be invalid: No
module named 'distutils.spawn'

2020-04-19 02:16:44,803 p=13395 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/oc.py) as it seems to be invalid: No
module named 'distutils.spawn'

2020-04-19 02:16:44,826 p=13395 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/zone.py) as it seems to be invalid: No
module named 'distutils.spawn'

2020-04-19 02:16:44,941 p=13395 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/callback/osx_say.py) as it seems to be invalid: No
module named 'distutils.spawn'

2020-04-19 02:16:44,942 p=13395 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/callback/say.py) as it seems to be invalid: No module
named 'distutils.spawn'

2020-04-19 02:16:44,968 p=13395 u=akokov |   [WARNING]: Could not match supplied host pattern, ignoring: test

2020-04-19 02:16:44,970 p=13395 u=akokov |  PLAY [test] **********************************************************************************************************************
2020-04-19 02:16:44,970 p=13395 u=akokov |  skipping: no hosts matched
2020-04-19 02:16:44,970 p=13395 u=akokov |  PLAY RECAP ***********************************************************************************************************************
2020-04-19 02:18:03,992 p=13577 u=akokov |   [WARNING]: provided hosts list is empty, only localhost is available. Note that the implicit localhost does not match 'all'

2020-04-19 02:18:04,003 p=13577 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/docker.py) as it seems to be invalid: No
module named 'distutils.spawn'

2020-04-19 02:18:04,036 p=13577 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/iocage.py) as it seems to be invalid: No
module named 'distutils.spawn'

2020-04-19 02:18:04,037 p=13577 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/jail.py) as it seems to be invalid: No
module named 'distutils.spawn'

2020-04-19 02:18:04,037 p=13577 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/kubectl.py) as it seems to be invalid: No
module named 'distutils.spawn'

2020-04-19 02:18:04,037 p=13577 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/libvirt_lxc.py) as it seems to be invalid:
No module named 'distutils.spawn'

2020-04-19 02:18:04,038 p=13577 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/lxd.py) as it seems to be invalid: No
module named 'distutils.spawn'

2020-04-19 02:18:04,044 p=13577 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/oc.py) as it seems to be invalid: No
module named 'distutils.spawn'

2020-04-19 02:18:04,068 p=13577 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/zone.py) as it seems to be invalid: No
module named 'distutils.spawn'

2020-04-19 02:18:04,184 p=13577 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/callback/osx_say.py) as it seems to be invalid: No
module named 'distutils.spawn'

2020-04-19 02:18:04,185 p=13577 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/callback/say.py) as it seems to be invalid: No module
named 'distutils.spawn'

2020-04-19 02:18:04,210 p=13577 u=akokov |   [WARNING]: Could not match supplied host pattern, ignoring: test

2020-04-19 02:18:04,211 p=13577 u=akokov |  PLAY [test] **********************************************************************************************************************
2020-04-19 02:18:04,212 p=13577 u=akokov |  skipping: no hosts matched
2020-04-19 02:18:04,212 p=13577 u=akokov |  PLAY RECAP ***********************************************************************************************************************
2020-04-19 02:18:08,488 p=13590 u=root |   [WARNING]: provided hosts list is empty, only localhost is available. Note that the implicit localhost does not match 'all'

2020-04-19 02:18:08,498 p=13590 u=root |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/docker.py) as it seems to be invalid: No
module named 'distutils.spawn'

2020-04-19 02:18:08,532 p=13590 u=root |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/iocage.py) as it seems to be invalid: No
module named 'distutils.spawn'

2020-04-19 02:18:08,532 p=13590 u=root |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/jail.py) as it seems to be invalid: No
module named 'distutils.spawn'

2020-04-19 02:18:08,532 p=13590 u=root |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/kubectl.py) as it seems to be invalid: No
module named 'distutils.spawn'

2020-04-19 02:18:08,533 p=13590 u=root |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/libvirt_lxc.py) as it seems to be invalid:
No module named 'distutils.spawn'

2020-04-19 02:18:08,535 p=13590 u=root |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/lxd.py) as it seems to be invalid: No
module named 'distutils.spawn'

2020-04-19 02:18:08,541 p=13590 u=root |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/oc.py) as it seems to be invalid: No
module named 'distutils.spawn'

2020-04-19 02:18:08,566 p=13590 u=root |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/zone.py) as it seems to be invalid: No
module named 'distutils.spawn'

2020-04-19 02:18:08,680 p=13590 u=root |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/callback/osx_say.py) as it seems to be invalid: No
module named 'distutils.spawn'

2020-04-19 02:18:08,681 p=13590 u=root |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/callback/say.py) as it seems to be invalid: No module
named 'distutils.spawn'

2020-04-19 02:18:08,707 p=13590 u=root |   [WARNING]: Could not match supplied host pattern, ignoring: test

2020-04-19 02:18:08,708 p=13590 u=root |  PLAY [test] **********************************************************************************************************************
2020-04-19 02:18:08,708 p=13590 u=root |  skipping: no hosts matched
2020-04-19 02:18:08,708 p=13590 u=root |  PLAY RECAP ***********************************************************************************************************************
2020-04-19 02:18:16,368 p=13617 u=root |  ansible-playbook 2.8.3
  config file = /home/ansible/ansible.cfg
  configured module search path = ['/root/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/lib/python3/dist-packages/ansible
  executable location = /usr/bin/ansible-playbook
  python version = 3.7.5 (default, Nov 20 2019, 09:21:52) [GCC 9.2.1 20191008]
2020-04-19 02:18:16,368 p=13617 u=root |  Using /home/ansible/ansible.cfg as config file
2020-04-19 02:18:16,369 p=13617 u=root |  setting up inventory plugins
2020-04-19 02:18:16,373 p=13617 u=root |  host_list declined parsing /etc/ansible/hosts as it did not pass it's verify_file() method
2020-04-19 02:18:16,373 p=13617 u=root |  script declined parsing /etc/ansible/hosts as it did not pass it's verify_file() method
2020-04-19 02:18:16,373 p=13617 u=root |  auto declined parsing /etc/ansible/hosts as it did not pass it's verify_file() method
2020-04-19 02:18:16,375 p=13617 u=root |  Parsed /etc/ansible/hosts inventory source with ini plugin
2020-04-19 02:18:16,375 p=13617 u=root |   [WARNING]: provided hosts list is empty, only localhost is available. Note that the implicit localhost does not match 'all'

2020-04-19 02:18:16,386 p=13617 u=root |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/docker.py) as it seems to be invalid: No
module named 'distutils.spawn'

2020-04-19 02:18:16,419 p=13617 u=root |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/iocage.py) as it seems to be invalid: No
module named 'distutils.spawn'

2020-04-19 02:18:16,419 p=13617 u=root |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/jail.py) as it seems to be invalid: No
module named 'distutils.spawn'

2020-04-19 02:18:16,419 p=13617 u=root |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/kubectl.py) as it seems to be invalid: No
module named 'distutils.spawn'

2020-04-19 02:18:16,419 p=13617 u=root |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/libvirt_lxc.py) as it seems to be invalid:
No module named 'distutils.spawn'

2020-04-19 02:18:16,421 p=13617 u=root |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/lxd.py) as it seems to be invalid: No
module named 'distutils.spawn'

2020-04-19 02:18:16,427 p=13617 u=root |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/oc.py) as it seems to be invalid: No
module named 'distutils.spawn'

2020-04-19 02:18:16,450 p=13617 u=root |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/zone.py) as it seems to be invalid: No
module named 'distutils.spawn'

2020-04-19 02:18:16,541 p=13617 u=root |  Loading callback plugin default of type stdout, v2.0 from /usr/lib/python3/dist-packages/ansible/plugins/callback/default.py
2020-04-19 02:18:16,564 p=13617 u=root |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/callback/osx_say.py) as it seems to be invalid: No
module named 'distutils.spawn'

2020-04-19 02:18:16,565 p=13617 u=root |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/callback/say.py) as it seems to be invalid: No module
named 'distutils.spawn'

2020-04-19 02:18:16,572 p=13617 u=root |  PLAYBOOK: playbook-install.yml ***************************************************************************************************
2020-04-19 02:18:16,572 p=13617 u=root |  1 plays in playbook-install.yml
2020-04-19 02:18:16,591 p=13617 u=root |   [WARNING]: Could not match supplied host pattern, ignoring: test

2020-04-19 02:18:16,592 p=13617 u=root |  PLAY [test] **********************************************************************************************************************
2020-04-19 02:18:16,593 p=13617 u=root |  skipping: no hosts matched
2020-04-19 02:18:16,593 p=13617 u=root |  PLAY RECAP ***********************************************************************************************************************
2020-04-19 02:19:05,690 p=13706 u=root |  Using /home/ansible/ansible.cfg as config file
2020-04-19 02:19:05,698 p=13706 u=root |   [WARNING]: provided hosts list is empty, only localhost is available. Note that the implicit localhost does not match 'all'

2020-04-19 02:19:05,707 p=13706 u=root |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/docker.py) as it seems to be invalid: No
module named 'distutils.spawn'

2020-04-19 02:19:05,741 p=13706 u=root |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/iocage.py) as it seems to be invalid: No
module named 'distutils.spawn'

2020-04-19 02:19:05,741 p=13706 u=root |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/jail.py) as it seems to be invalid: No
module named 'distutils.spawn'

2020-04-19 02:19:05,741 p=13706 u=root |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/kubectl.py) as it seems to be invalid: No
module named 'distutils.spawn'

2020-04-19 02:19:05,742 p=13706 u=root |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/libvirt_lxc.py) as it seems to be invalid:
No module named 'distutils.spawn'

2020-04-19 02:19:05,743 p=13706 u=root |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/lxd.py) as it seems to be invalid: No
module named 'distutils.spawn'

2020-04-19 02:19:05,750 p=13706 u=root |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/oc.py) as it seems to be invalid: No
module named 'distutils.spawn'

2020-04-19 02:19:05,772 p=13706 u=root |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/zone.py) as it seems to be invalid: No
module named 'distutils.spawn'

2020-04-19 02:19:05,888 p=13706 u=root |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/callback/osx_say.py) as it seems to be invalid: No
module named 'distutils.spawn'

2020-04-19 02:19:05,889 p=13706 u=root |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/callback/say.py) as it seems to be invalid: No module
named 'distutils.spawn'

2020-04-19 02:19:05,916 p=13706 u=root |   [WARNING]: Could not match supplied host pattern, ignoring: test

2020-04-19 02:19:05,918 p=13706 u=root |  PLAY [test] **********************************************************************************************************************
2020-04-19 02:19:05,918 p=13706 u=root |  skipping: no hosts matched
2020-04-19 02:19:05,918 p=13706 u=root |  PLAY RECAP ***********************************************************************************************************************
2020-04-19 02:19:08,919 p=13722 u=root |  ansible-playbook 2.8.3
  config file = /home/ansible/ansible.cfg
  configured module search path = ['/root/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/lib/python3/dist-packages/ansible
  executable location = /usr/bin/ansible-playbook
  python version = 3.7.5 (default, Nov 20 2019, 09:21:52) [GCC 9.2.1 20191008]
2020-04-19 02:19:08,919 p=13722 u=root |  Using /home/ansible/ansible.cfg as config file
2020-04-19 02:19:08,919 p=13722 u=root |  setting up inventory plugins
2020-04-19 02:19:08,923 p=13722 u=root |  host_list declined parsing /etc/ansible/hosts as it did not pass it's verify_file() method
2020-04-19 02:19:08,923 p=13722 u=root |  script declined parsing /etc/ansible/hosts as it did not pass it's verify_file() method
2020-04-19 02:19:08,924 p=13722 u=root |  auto declined parsing /etc/ansible/hosts as it did not pass it's verify_file() method
2020-04-19 02:19:08,925 p=13722 u=root |  Parsed /etc/ansible/hosts inventory source with ini plugin
2020-04-19 02:19:08,926 p=13722 u=root |   [WARNING]: provided hosts list is empty, only localhost is available. Note that the implicit localhost does not match 'all'

2020-04-19 02:19:08,935 p=13722 u=root |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/docker.py) as it seems to be invalid: No
module named 'distutils.spawn'

2020-04-19 02:19:08,969 p=13722 u=root |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/iocage.py) as it seems to be invalid: No
module named 'distutils.spawn'

2020-04-19 02:19:08,969 p=13722 u=root |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/jail.py) as it seems to be invalid: No
module named 'distutils.spawn'

2020-04-19 02:19:08,969 p=13722 u=root |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/kubectl.py) as it seems to be invalid: No
module named 'distutils.spawn'

2020-04-19 02:19:08,970 p=13722 u=root |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/libvirt_lxc.py) as it seems to be invalid:
No module named 'distutils.spawn'

2020-04-19 02:19:08,971 p=13722 u=root |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/lxd.py) as it seems to be invalid: No
module named 'distutils.spawn'

2020-04-19 02:19:08,977 p=13722 u=root |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/oc.py) as it seems to be invalid: No
module named 'distutils.spawn'

2020-04-19 02:19:09,001 p=13722 u=root |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/zone.py) as it seems to be invalid: No
module named 'distutils.spawn'

2020-04-19 02:19:09,093 p=13722 u=root |  Loading callback plugin default of type stdout, v2.0 from /usr/lib/python3/dist-packages/ansible/plugins/callback/default.py
2020-04-19 02:19:09,116 p=13722 u=root |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/callback/osx_say.py) as it seems to be invalid: No
module named 'distutils.spawn'

2020-04-19 02:19:09,117 p=13722 u=root |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/callback/say.py) as it seems to be invalid: No module
named 'distutils.spawn'

2020-04-19 02:19:09,124 p=13722 u=root |  PLAYBOOK: playbook-install.yml ***************************************************************************************************
2020-04-19 02:19:09,124 p=13722 u=root |  1 plays in playbook-install.yml
2020-04-19 02:19:09,144 p=13722 u=root |   [WARNING]: Could not match supplied host pattern, ignoring: test

2020-04-19 02:19:09,145 p=13722 u=root |  PLAY [test] **********************************************************************************************************************
2020-04-19 02:19:09,145 p=13722 u=root |  skipping: no hosts matched
2020-04-19 02:19:09,145 p=13722 u=root |  PLAY RECAP ***********************************************************************************************************************
2020-04-19 02:23:49,231 p=13911 u=akokov |   [WARNING]: provided hosts list is empty, only localhost is available. Note that the implicit localhost does not match 'all'

2020-04-19 02:23:49,241 p=13911 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/docker.py) as it seems to be invalid: No
module named 'distutils.spawn'

2020-04-19 02:23:49,273 p=13911 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/iocage.py) as it seems to be invalid: No
module named 'distutils.spawn'

2020-04-19 02:23:49,273 p=13911 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/jail.py) as it seems to be invalid: No
module named 'distutils.spawn'

2020-04-19 02:23:49,273 p=13911 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/kubectl.py) as it seems to be invalid: No
module named 'distutils.spawn'

2020-04-19 02:23:49,274 p=13911 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/libvirt_lxc.py) as it seems to be invalid:
No module named 'distutils.spawn'

2020-04-19 02:23:49,275 p=13911 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/lxd.py) as it seems to be invalid: No
module named 'distutils.spawn'

2020-04-19 02:23:49,281 p=13911 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/oc.py) as it seems to be invalid: No
module named 'distutils.spawn'

2020-04-19 02:23:49,303 p=13911 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/zone.py) as it seems to be invalid: No
module named 'distutils.spawn'

2020-04-19 02:23:49,419 p=13911 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/callback/osx_say.py) as it seems to be invalid: No
module named 'distutils.spawn'

2020-04-19 02:23:49,420 p=13911 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/callback/say.py) as it seems to be invalid: No module
named 'distutils.spawn'

2020-04-19 02:23:49,446 p=13911 u=akokov |   [WARNING]: Could not match supplied host pattern, ignoring: test

2020-04-19 02:23:49,447 p=13911 u=akokov |  PLAY [test] **********************************************************************************************************************
2020-04-19 02:23:49,447 p=13911 u=akokov |  skipping: no hosts matched
2020-04-19 02:23:49,447 p=13911 u=akokov |  PLAY RECAP ***********************************************************************************************************************
2020-04-19 02:23:53,356 p=13926 u=akokov |  ansible-playbook 2.8.3
  config file = /home/ansible/ansible.cfg
  configured module search path = ['/home/akokov/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/lib/python3/dist-packages/ansible
  executable location = /usr/bin/ansible-playbook
  python version = 3.7.5 (default, Nov 20 2019, 09:21:52) [GCC 9.2.1 20191008]
2020-04-19 02:23:53,356 p=13926 u=akokov |  Using /home/ansible/ansible.cfg as config file
2020-04-19 02:23:53,357 p=13926 u=akokov |  setting up inventory plugins
2020-04-19 02:23:53,361 p=13926 u=akokov |  host_list declined parsing /etc/ansible/hosts as it did not pass it's verify_file() method
2020-04-19 02:23:53,361 p=13926 u=akokov |  script declined parsing /etc/ansible/hosts as it did not pass it's verify_file() method
2020-04-19 02:23:53,361 p=13926 u=akokov |  auto declined parsing /etc/ansible/hosts as it did not pass it's verify_file() method
2020-04-19 02:23:53,363 p=13926 u=akokov |  Parsed /etc/ansible/hosts inventory source with ini plugin
2020-04-19 02:23:53,363 p=13926 u=akokov |   [WARNING]: provided hosts list is empty, only localhost is available. Note that the implicit localhost does not match 'all'

2020-04-19 02:23:53,372 p=13926 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/docker.py) as it seems to be invalid: No
module named 'distutils.spawn'

2020-04-19 02:23:53,404 p=13926 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/iocage.py) as it seems to be invalid: No
module named 'distutils.spawn'

2020-04-19 02:23:53,404 p=13926 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/jail.py) as it seems to be invalid: No
module named 'distutils.spawn'

2020-04-19 02:23:53,405 p=13926 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/kubectl.py) as it seems to be invalid: No
module named 'distutils.spawn'

2020-04-19 02:23:53,405 p=13926 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/libvirt_lxc.py) as it seems to be invalid:
No module named 'distutils.spawn'

2020-04-19 02:23:53,406 p=13926 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/lxd.py) as it seems to be invalid: No
module named 'distutils.spawn'

2020-04-19 02:23:53,412 p=13926 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/oc.py) as it seems to be invalid: No
module named 'distutils.spawn'

2020-04-19 02:23:53,434 p=13926 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/zone.py) as it seems to be invalid: No
module named 'distutils.spawn'

2020-04-19 02:23:53,524 p=13926 u=akokov |  Loading callback plugin default of type stdout, v2.0 from /usr/lib/python3/dist-packages/ansible/plugins/callback/default.py
2020-04-19 02:23:53,546 p=13926 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/callback/osx_say.py) as it seems to be invalid: No
module named 'distutils.spawn'

2020-04-19 02:23:53,547 p=13926 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/callback/say.py) as it seems to be invalid: No module
named 'distutils.spawn'

2020-04-19 02:23:53,554 p=13926 u=akokov |  PLAYBOOK: playbook-install.yml ***************************************************************************************************
2020-04-19 02:23:53,555 p=13926 u=akokov |  1 plays in playbook-install.yml
2020-04-19 02:23:53,573 p=13926 u=akokov |   [WARNING]: Could not match supplied host pattern, ignoring: test

2020-04-19 02:23:53,574 p=13926 u=akokov |  PLAY [test] **********************************************************************************************************************
2020-04-19 02:23:53,574 p=13926 u=akokov |  skipping: no hosts matched
2020-04-19 02:23:53,574 p=13926 u=akokov |  PLAY RECAP ***********************************************************************************************************************
2020-04-19 02:25:29,300 p=14143 u=akokov |  ERROR! the playbook: l could not be found
2020-04-19 02:25:33,098 p=14151 u=akokov |  ansible-playbook 2.8.3
  config file = /home/ansible/ansible.cfg
  configured module search path = ['/home/akokov/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/lib/python3/dist-packages/ansible
  executable location = /usr/bin/ansible-playbook
  python version = 3.7.5 (default, Nov 20 2019, 09:21:52) [GCC 9.2.1 20191008]
2020-04-19 02:25:33,098 p=14151 u=akokov |  Using /home/ansible/ansible.cfg as config file
2020-04-19 02:25:33,099 p=14151 u=akokov |  setting up inventory plugins
2020-04-19 02:25:33,103 p=14151 u=akokov |  host_list declined parsing /etc/ansible/hosts as it did not pass it's verify_file() method
2020-04-19 02:25:33,103 p=14151 u=akokov |  script declined parsing /etc/ansible/hosts as it did not pass it's verify_file() method
2020-04-19 02:25:33,103 p=14151 u=akokov |  auto declined parsing /etc/ansible/hosts as it did not pass it's verify_file() method
2020-04-19 02:25:33,105 p=14151 u=akokov |  Parsed /etc/ansible/hosts inventory source with ini plugin
2020-04-19 02:25:33,105 p=14151 u=akokov |   [WARNING]: provided hosts list is empty, only localhost is available. Note that the implicit localhost does not match 'all'

2020-04-19 02:25:33,114 p=14151 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/docker.py) as it seems to be invalid: No module named 'distutils.spawn'

2020-04-19 02:25:33,146 p=14151 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/iocage.py) as it seems to be invalid: No module named 'distutils.spawn'

2020-04-19 02:25:33,146 p=14151 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/jail.py) as it seems to be invalid: No module named 'distutils.spawn'

2020-04-19 02:25:33,147 p=14151 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/kubectl.py) as it seems to be invalid: No module named 'distutils.spawn'

2020-04-19 02:25:33,147 p=14151 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/libvirt_lxc.py) as it seems to be invalid: No module named 'distutils.spawn'

2020-04-19 02:25:33,148 p=14151 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/lxd.py) as it seems to be invalid: No module named 'distutils.spawn'

2020-04-19 02:25:33,154 p=14151 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/oc.py) as it seems to be invalid: No module named 'distutils.spawn'

2020-04-19 02:25:33,177 p=14151 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/zone.py) as it seems to be invalid: No module named 'distutils.spawn'

2020-04-19 02:25:33,265 p=14151 u=akokov |  Loading callback plugin default of type stdout, v2.0 from /usr/lib/python3/dist-packages/ansible/plugins/callback/default.py
2020-04-19 02:25:33,287 p=14151 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/callback/osx_say.py) as it seems to be invalid: No module named 'distutils.spawn'

2020-04-19 02:25:33,288 p=14151 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/callback/say.py) as it seems to be invalid: No module named 'distutils.spawn'

2020-04-19 02:25:33,295 p=14151 u=akokov |  PLAYBOOK: playbook-install.yml **************************************************************************************************************************************************************************************
2020-04-19 02:25:33,295 p=14151 u=akokov |  1 plays in playbook-install.yml
2020-04-19 02:25:33,314 p=14151 u=akokov |   [WARNING]: Could not match supplied host pattern, ignoring: test

2020-04-19 02:25:33,315 p=14151 u=akokov |  PLAY [test] *********************************************************************************************************************************************************************************************************
2020-04-19 02:25:33,315 p=14151 u=akokov |  skipping: no hosts matched
2020-04-19 02:25:33,315 p=14151 u=akokov |  PLAY RECAP **********************************************************************************************************************************************************************************************************
2020-04-19 02:25:55,284 p=14199 u=akokov |  ansible-playbook 2.8.3
  config file = /home/ansible/ansible.cfg
  configured module search path = ['/home/akokov/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/lib/python3/dist-packages/ansible
  executable location = /usr/bin/ansible-playbook
  python version = 3.7.5 (default, Nov 20 2019, 09:21:52) [GCC 9.2.1 20191008]
2020-04-19 02:25:55,285 p=14199 u=akokov |  Using /home/ansible/ansible.cfg as config file
2020-04-19 02:25:55,285 p=14199 u=akokov |  setting up inventory plugins
2020-04-19 02:25:55,289 p=14199 u=akokov |  host_list declined parsing /etc/ansible/hosts as it did not pass it's verify_file() method
2020-04-19 02:25:55,289 p=14199 u=akokov |  script declined parsing /etc/ansible/hosts as it did not pass it's verify_file() method
2020-04-19 02:25:55,289 p=14199 u=akokov |  auto declined parsing /etc/ansible/hosts as it did not pass it's verify_file() method
2020-04-19 02:25:55,291 p=14199 u=akokov |  Parsed /etc/ansible/hosts inventory source with ini plugin
2020-04-19 02:25:55,291 p=14199 u=akokov |   [WARNING]: provided hosts list is empty, only localhost is available. Note that the implicit localhost does not match 'all'

2020-04-19 02:25:55,300 p=14199 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/docker.py) as it seems to be invalid: No module named 'distutils.spawn'

2020-04-19 02:25:55,332 p=14199 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/iocage.py) as it seems to be invalid: No module named 'distutils.spawn'

2020-04-19 02:25:55,332 p=14199 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/jail.py) as it seems to be invalid: No module named 'distutils.spawn'

2020-04-19 02:25:55,332 p=14199 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/kubectl.py) as it seems to be invalid: No module named 'distutils.spawn'

2020-04-19 02:25:55,333 p=14199 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/libvirt_lxc.py) as it seems to be invalid: No module named 'distutils.spawn'

2020-04-19 02:25:55,334 p=14199 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/lxd.py) as it seems to be invalid: No module named 'distutils.spawn'

2020-04-19 02:25:55,340 p=14199 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/oc.py) as it seems to be invalid: No module named 'distutils.spawn'

2020-04-19 02:25:55,362 p=14199 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/zone.py) as it seems to be invalid: No module named 'distutils.spawn'

2020-04-19 02:25:55,450 p=14199 u=akokov |  Loading callback plugin default of type stdout, v2.0 from /usr/lib/python3/dist-packages/ansible/plugins/callback/default.py
2020-04-19 02:25:55,471 p=14199 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/callback/osx_say.py) as it seems to be invalid: No module named 'distutils.spawn'

2020-04-19 02:25:55,472 p=14199 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/callback/say.py) as it seems to be invalid: No module named 'distutils.spawn'

2020-04-19 02:25:55,479 p=14199 u=akokov |  PLAYBOOK: playbook-install.yml **************************************************************************************************************************************************************************************
2020-04-19 02:25:55,480 p=14199 u=akokov |  1 plays in playbook-install.yml
2020-04-19 02:25:55,498 p=14199 u=akokov |   [WARNING]: Could not match supplied host pattern, ignoring: test

2020-04-19 02:25:55,499 p=14199 u=akokov |  PLAY [test] *********************************************************************************************************************************************************************************************************
2020-04-19 02:25:55,499 p=14199 u=akokov |  skipping: no hosts matched
2020-04-19 02:25:55,499 p=14199 u=akokov |  PLAY RECAP **********************************************************************************************************************************************************************************************************
2020-04-19 02:28:43,327 p=14441 u=akokov |  ansible-playbook 2.8.3
  config file = /home/ansible/ansible.cfg
  configured module search path = ['/home/akokov/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/lib/python3/dist-packages/ansible
  executable location = /usr/bin/ansible-playbook
  python version = 3.7.5 (default, Nov 20 2019, 09:21:52) [GCC 9.2.1 20191008]
2020-04-19 02:28:43,327 p=14441 u=akokov |  Using /home/ansible/ansible.cfg as config file
2020-04-19 02:28:43,328 p=14441 u=akokov |  setting up inventory plugins
2020-04-19 02:28:43,332 p=14441 u=akokov |  host_list declined parsing /etc/ansible/hosts as it did not pass it's verify_file() method
2020-04-19 02:28:43,332 p=14441 u=akokov |  script declined parsing /etc/ansible/hosts as it did not pass it's verify_file() method
2020-04-19 02:28:43,332 p=14441 u=akokov |  auto declined parsing /etc/ansible/hosts as it did not pass it's verify_file() method
2020-04-19 02:28:43,334 p=14441 u=akokov |  Parsed /etc/ansible/hosts inventory source with ini plugin
2020-04-19 02:28:43,334 p=14441 u=akokov |   [WARNING]: provided hosts list is empty, only localhost is available. Note that the implicit localhost does not match 'all'

2020-04-19 02:28:43,344 p=14441 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/docker.py) as it seems to be invalid: No module named 'distutils.spawn'

2020-04-19 02:28:43,378 p=14441 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/iocage.py) as it seems to be invalid: No module named 'distutils.spawn'

2020-04-19 02:28:43,378 p=14441 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/jail.py) as it seems to be invalid: No module named 'distutils.spawn'

2020-04-19 02:28:43,378 p=14441 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/kubectl.py) as it seems to be invalid: No module named 'distutils.spawn'

2020-04-19 02:28:43,379 p=14441 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/libvirt_lxc.py) as it seems to be invalid: No module named 'distutils.spawn'

2020-04-19 02:28:43,380 p=14441 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/lxd.py) as it seems to be invalid: No module named 'distutils.spawn'

2020-04-19 02:28:43,386 p=14441 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/oc.py) as it seems to be invalid: No module named 'distutils.spawn'

2020-04-19 02:28:43,413 p=14441 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/zone.py) as it seems to be invalid: No module named 'distutils.spawn'

2020-04-19 02:28:43,504 p=14441 u=akokov |  Loading callback plugin default of type stdout, v2.0 from /usr/lib/python3/dist-packages/ansible/plugins/callback/default.py
2020-04-19 02:28:43,526 p=14441 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/callback/osx_say.py) as it seems to be invalid: No module named 'distutils.spawn'

2020-04-19 02:28:43,527 p=14441 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/callback/say.py) as it seems to be invalid: No module named 'distutils.spawn'

2020-04-19 02:28:43,534 p=14441 u=akokov |  PLAYBOOK: playbook-install.yml **************************************************************************************************************************************************************************************
2020-04-19 02:28:43,535 p=14441 u=akokov |  1 plays in playbook-install.yml
2020-04-19 02:28:43,553 p=14441 u=akokov |   [WARNING]: Could not match supplied host pattern, ignoring: test

2020-04-19 02:28:43,554 p=14441 u=akokov |  PLAY [test] *********************************************************************************************************************************************************************************************************
2020-04-19 02:28:43,554 p=14441 u=akokov |  skipping: no hosts matched
2020-04-19 02:28:43,554 p=14441 u=akokov |  PLAY RECAP **********************************************************************************************************************************************************************************************************
2020-04-19 02:29:05,265 p=14491 u=akokov |  ansible-playbook 2.8.3
  config file = /home/ansible/ansible.cfg
  configured module search path = ['/home/akokov/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/lib/python3/dist-packages/ansible
  executable location = /usr/bin/ansible-playbook
  python version = 3.7.5 (default, Nov 20 2019, 09:21:52) [GCC 9.2.1 20191008]
2020-04-19 02:29:05,265 p=14491 u=akokov |  Using /home/ansible/ansible.cfg as config file
2020-04-19 02:29:05,266 p=14491 u=akokov |  setting up inventory plugins
2020-04-19 02:29:05,288 p=14491 u=akokov |  host_list declined parsing /home/ansible/hosts as it did not pass it's verify_file() method
2020-04-19 02:29:05,298 p=14491 u=akokov |  auto declined parsing /home/ansible/hosts as it did not pass it's verify_file() method
2020-04-19 02:29:05,307 p=14491 u=akokov |  Parsed /home/ansible/hosts inventory source with ini plugin
2020-04-19 02:29:05,354 p=14491 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/docker.py) as it seems to be invalid: No module named 'distutils.spawn'

2020-04-19 02:29:05,406 p=14491 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/iocage.py) as it seems to be invalid: No module named 'distutils.spawn'

2020-04-19 02:29:05,407 p=14491 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/jail.py) as it seems to be invalid: No module named 'distutils.spawn'

2020-04-19 02:29:05,407 p=14491 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/kubectl.py) as it seems to be invalid: No module named 'distutils.spawn'

2020-04-19 02:29:05,407 p=14491 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/libvirt_lxc.py) as it seems to be invalid: No module named 'distutils.spawn'

2020-04-19 02:29:05,408 p=14491 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/lxd.py) as it seems to be invalid: No module named 'distutils.spawn'

2020-04-19 02:29:05,414 p=14491 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/oc.py) as it seems to be invalid: No module named 'distutils.spawn'

2020-04-19 02:29:05,436 p=14491 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/zone.py) as it seems to be invalid: No module named 'distutils.spawn'

2020-04-19 02:29:05,529 p=14491 u=akokov |  Loading callback plugin default of type stdout, v2.0 from /usr/lib/python3/dist-packages/ansible/plugins/callback/default.py
2020-04-19 02:29:05,551 p=14491 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/callback/osx_say.py) as it seems to be invalid: No module named 'distutils.spawn'

2020-04-19 02:29:05,552 p=14491 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/callback/say.py) as it seems to be invalid: No module named 'distutils.spawn'

2020-04-19 02:29:05,559 p=14491 u=akokov |  PLAYBOOK: playbook-install.yml **************************************************************************************************************************************************************************************
2020-04-19 02:29:05,559 p=14491 u=akokov |  1 plays in playbook-install.yml
2020-04-19 02:29:05,580 p=14491 u=akokov |  PLAY [test] *********************************************************************************************************************************************************************************************************
2020-04-19 02:29:05,585 p=14491 u=akokov |  TASK [Gathering Facts] **********************************************************************************************************************************************************************************************
2020-04-19 02:29:05,585 p=14491 u=akokov |  task path: /home/ansible/playbook-install.yml:3
2020-04-19 02:29:05,593 p=14491 u=akokov |  <192.168.1.6> ESTABLISH SSH CONNECTION FOR USER: root
2020-04-19 02:29:05,594 p=14491 u=akokov |  <192.168.1.6> SSH: EXEC ssh -vvv -C -o ControlMaster=auto -o ControlPersist=60s -o KbdInteractiveAuthentication=no -o PreferredAuthentications=gssapi-with-mic,gssapi-keyex,hostbased,publickey -o PasswordAuthentication=no -o 'User="root"' -o ConnectTimeout=10 -o ControlPath=/home/akokov/.ansible/cp/c3e3deaa98 192.168.1.6 '/bin/sh -c '"'"'echo ~root && sleep 0'"'"''
2020-04-19 02:29:06,933 p=14491 u=akokov |  <192.168.1.6> (0, b'/root\n', b'OpenSSH_8.0p1 Ubuntu-6build1, OpenSSL 1.1.1c  28 May 2019\r\ndebug1: Reading configuration data /etc/ssh/ssh_config\r\ndebug1: /etc/ssh/ssh_config line 19: Applying options for *\r\ndebug2: resolve_canonicalize: hostname 192.168.1.6 is address\r\ndebug1: auto-mux: Trying existing master\r\ndebug1: Control socket "/home/akokov/.ansible/cp/c3e3deaa98" does not exist\r\ndebug2: ssh_connect_direct\r\ndebug1: Connecting to 192.168.1.6 [192.168.1.6] port 22.\r\ndebug2: fd 3 setting O_NONBLOCK\r\ndebug1: fd 3 clearing O_NONBLOCK\r\ndebug1: Connection established.\r\ndebug3: timeout: 9992 ms remain after connect\r\ndebug1: identity file /home/akokov/.ssh/id_rsa type 0\r\ndebug1: identity file /home/akokov/.ssh/id_rsa-cert type -1\r\ndebug1: identity file /home/akokov/.ssh/id_dsa type -1\r\ndebug1: identity file /home/akokov/.ssh/id_dsa-cert type -1\r\ndebug1: identity file /home/akokov/.ssh/id_ecdsa type -1\r\ndebug1: identity file /home/akokov/.ssh/id_ecdsa-cert type -1\r\ndebug1: identity file /home/akokov/.ssh/id_ed25519 type -1\r\ndebug1: identity file /home/akokov/.ssh/id_ed25519-cert type -1\r\ndebug1: identity file /home/akokov/.ssh/id_xmss type -1\r\ndebug1: identity file /home/akokov/.ssh/id_xmss-cert type -1\r\ndebug1: Local version string SSH-2.0-OpenSSH_8.0p1 Ubuntu-6build1\r\ndebug1: Remote protocol version 2.0, remote software version OpenSSH_8.0\r\ndebug1: match: OpenSSH_8.0 pat OpenSSH* compat 0x04000000\r\ndebug2: fd 3 setting O_NONBLOCK\r\ndebug1: Authenticating to 192.168.1.6:22 as \'root\'\r\ndebug3: hostkeys_foreach: reading file "/home/akokov/.ssh/known_hosts"\r\ndebug3: record_hostkey: found key type ECDSA in file /home/akokov/.ssh/known_hosts:9\r\ndebug3: load_hostkeys: loaded 1 keys from 192.168.1.6\r\ndebug3: order_hostkeyalgs: prefer hostkeyalgs: ecdsa-sha2-nistp256-cert-v01@openssh.com,ecdsa-sha2-nistp384-cert-v01@openssh.com,ecdsa-sha2-nistp521-cert-v01@openssh.com,ecdsa-sha2-nistp256,ecdsa-sha2-nistp384,ecdsa-sha2-nistp521\r\ndebug3: send packet: type 20\r\ndebug1: SSH2_MSG_KEXINIT sent\r\ndebug3: receive packet: type 20\r\ndebug1: SSH2_MSG_KEXINIT received\r\ndebug2: local client KEXINIT proposal\r\ndebug2: KEX algorithms: curve25519-sha256,curve25519-sha256@libssh.org,ecdh-sha2-nistp256,ecdh-sha2-nistp384,ecdh-sha2-nistp521,diffie-hellman-group-exchange-sha256,diffie-hellman-group16-sha512,diffie-hellman-group18-sha512,diffie-hellman-group14-sha256,diffie-hellman-group14-sha1,ext-info-c\r\ndebug2: host key algorithms: ecdsa-sha2-nistp256-cert-v01@openssh.com,ecdsa-sha2-nistp384-cert-v01@openssh.com,ecdsa-sha2-nistp521-cert-v01@openssh.com,ecdsa-sha2-nistp256,ecdsa-sha2-nistp384,ecdsa-sha2-nistp521,ssh-ed25519-cert-v01@openssh.com,rsa-sha2-512-cert-v01@openssh.com,rsa-sha2-256-cert-v01@openssh.com,ssh-rsa-cert-v01@openssh.com,ssh-ed25519,rsa-sha2-512,rsa-sha2-256,ssh-rsa\r\ndebug2: ciphers ctos: chacha20-poly1305@openssh.com,aes128-ctr,aes192-ctr,aes256-ctr,aes128-gcm@openssh.com,aes256-gcm@openssh.com\r\ndebug2: ciphers stoc: chacha20-poly1305@openssh.com,aes128-ctr,aes192-ctr,aes256-ctr,aes128-gcm@openssh.com,aes256-gcm@openssh.com\r\ndebug2: MACs ctos: umac-64-etm@openssh.com,umac-128-etm@openssh.com,hmac-sha2-256-etm@openssh.com,hmac-sha2-512-etm@openssh.com,hmac-sha1-etm@openssh.com,umac-64@openssh.com,umac-128@openssh.com,hmac-sha2-256,hmac-sha2-512,hmac-sha1\r\ndebug2: MACs stoc: umac-64-etm@openssh.com,umac-128-etm@openssh.com,hmac-sha2-256-etm@openssh.com,hmac-sha2-512-etm@openssh.com,hmac-sha1-etm@openssh.com,umac-64@openssh.com,umac-128@openssh.com,hmac-sha2-256,hmac-sha2-512,hmac-sha1\r\ndebug2: compression ctos: zlib@openssh.com,zlib,none\r\ndebug2: compression stoc: zlib@openssh.com,zlib,none\r\ndebug2: languages ctos: \r\ndebug2: languages stoc: \r\ndebug2: first_kex_follows 0 \r\ndebug2: reserved 0 \r\ndebug2: peer server KEXINIT proposal\r\ndebug2: KEX algorithms: curve25519-sha256,curve25519-sha256@libssh.org,ecdh-sha2-nistp256,ecdh-sha2-nistp384,ecdh-sha2-nistp521,diffie-hellman-group-exchange-sha256,diffie-hellman-group14-sha256,diffie-hellman-group16-sha512,diffie-hellman-group18-sha512,diffie-hellman-group-exchange-sha1,diffie-hellman-group14-sha1\r\ndebug2: host key algorithms: rsa-sha2-512,rsa-sha2-256,ssh-rsa,ecdsa-sha2-nistp256,ssh-ed25519\r\ndebug2: ciphers ctos: aes256-gcm@openssh.com,chacha20-poly1305@openssh.com,aes256-ctr,aes256-cbc,aes128-gcm@openssh.com,aes128-ctr,aes128-cbc\r\ndebug2: ciphers stoc: aes256-gcm@openssh.com,chacha20-poly1305@openssh.com,aes256-ctr,aes256-cbc,aes128-gcm@openssh.com,aes128-ctr,aes128-cbc\r\ndebug2: MACs ctos: hmac-sha2-256-etm@openssh.com,hmac-sha1-etm@openssh.com,umac-128-etm@openssh.com,hmac-sha2-512-etm@openssh.com,hmac-sha2-256,hmac-sha1,umac-128@openssh.com,hmac-sha2-512\r\ndebug2: MACs stoc: hmac-sha2-256-etm@openssh.com,hmac-sha1-etm@openssh.com,umac-128-etm@openssh.com,hmac-sha2-512-etm@openssh.com,hmac-sha2-256,hmac-sha1,umac-128@openssh.com,hmac-sha2-512\r\ndebug2: compression ctos: none,zlib@openssh.com\r\ndebug2: compression stoc: none,zlib@openssh.com\r\ndebug2: languages ctos: \r\ndebug2: languages stoc: \r\ndebug2: first_kex_follows 0 \r\ndebug2: reserved 0 \r\ndebug1: kex: algorithm: curve25519-sha256\r\ndebug1: kex: host key algorithm: ecdsa-sha2-nistp256\r\ndebug1: kex: server->client cipher: chacha20-poly1305@openssh.com MAC: <implicit> compression: zlib@openssh.com\r\ndebug1: kex: client->server cipher: chacha20-poly1305@openssh.com MAC: <implicit> compression: zlib@openssh.com\r\ndebug3: send packet: type 30\r\ndebug1: expecting SSH2_MSG_KEX_ECDH_REPLY\r\ndebug3: receive packet: type 31\r\ndebug1: Server host key: ecdsa-sha2-nistp256 SHA256:SdE0z6VZBmj0Bxulj8g/vS+LjIVnBcsw60nxtz5o4MY\r\ndebug3: hostkeys_foreach: reading file "/home/akokov/.ssh/known_hosts"\r\ndebug3: record_hostkey: found key type ECDSA in file /home/akokov/.ssh/known_hosts:9\r\ndebug3: load_hostkeys: loaded 1 keys from 192.168.1.6\r\ndebug1: Host \'192.168.1.6\' is known and matches the ECDSA host key.\r\ndebug1: Found key in /home/akokov/.ssh/known_hosts:9\r\ndebug3: send packet: type 21\r\ndebug2: set_newkeys: mode 1\r\ndebug1: rekey out after 134217728 blocks\r\ndebug1: SSH2_MSG_NEWKEYS sent\r\ndebug1: expecting SSH2_MSG_NEWKEYS\r\ndebug3: receive packet: type 21\r\ndebug1: SSH2_MSG_NEWKEYS received\r\ndebug2: set_newkeys: mode 0\r\ndebug1: rekey in after 134217728 blocks\r\ndebug1: Will attempt key: /home/akokov/.ssh/id_rsa RSA SHA256:ZVMDLUmwxSE91kA0BMMWEJem9BeOMzEacGhQYM5wfkQ agent\r\ndebug1: Will attempt key: /home/akokov/.ssh/id_dsa \r\ndebug1: Will attempt key: /home/akokov/.ssh/id_ecdsa \r\ndebug1: Will attempt key: /home/akokov/.ssh/id_ed25519 \r\ndebug1: Will attempt key: /home/akokov/.ssh/id_xmss \r\ndebug2: pubkey_prepare: done\r\ndebug3: send packet: type 5\r\ndebug3: receive packet: type 7\r\ndebug1: SSH2_MSG_EXT_INFO received\r\ndebug1: kex_input_ext_info: server-sig-algs=<ssh-ed25519,ssh-rsa,rsa-sha2-256,rsa-sha2-512,ssh-dss,ecdsa-sha2-nistp256,ecdsa-sha2-nistp384,ecdsa-sha2-nistp521>\r\ndebug3: receive packet: type 6\r\ndebug2: service_accept: ssh-userauth\r\ndebug1: SSH2_MSG_SERVICE_ACCEPT received\r\ndebug3: send packet: type 50\r\ndebug3: receive packet: type 51\r\ndebug1: Authentications that can continue: publickey,gssapi-keyex,gssapi-with-mic,password\r\ndebug3: start over, passed a different list publickey,gssapi-keyex,gssapi-with-mic,password\r\ndebug3: preferred gssapi-with-mic,gssapi-keyex,hostbased,publickey\r\ndebug3: authmethod_lookup gssapi-with-mic\r\ndebug3: remaining preferred: gssapi-keyex,hostbased,publickey\r\ndebug3: authmethod_is_enabled gssapi-with-mic\r\ndebug1: Next authentication method: gssapi-with-mic\r\ndebug1: Unspecified GSS failure.  Minor code may provide more information\nNo Kerberos credentials available (default cache: FILE:/tmp/krb5cc_1000)\n\n\r\ndebug1: Unspecified GSS failure.  Minor code may provide more information\nNo Kerberos credentials available (default cache: FILE:/tmp/krb5cc_1000)\n\n\r\ndebug2: we did not send a packet, disable method\r\ndebug3: authmethod_lookup gssapi-keyex\r\ndebug3: remaining preferred: hostbased,publickey\r\ndebug3: authmethod_lookup publickey\r\ndebug3: remaining preferred: ,publickey\r\ndebug3: authmethod_is_enabled publickey\r\ndebug1: Next authentication method: publickey\r\ndebug1: Offering public key: /home/akokov/.ssh/id_rsa RSA SHA256:ZVMDLUmwxSE91kA0BMMWEJem9BeOMzEacGhQYM5wfkQ agent\r\ndebug3: send packet: type 50\r\ndebug2: we sent a publickey packet, wait for reply\r\ndebug3: receive packet: type 60\r\ndebug1: Server accepts key: /home/akokov/.ssh/id_rsa RSA SHA256:ZVMDLUmwxSE91kA0BMMWEJem9BeOMzEacGhQYM5wfkQ agent\r\ndebug3: sign_and_send_pubkey: RSA SHA256:ZVMDLUmwxSE91kA0BMMWEJem9BeOMzEacGhQYM5wfkQ\r\ndebug3: sign_and_send_pubkey: signing using rsa-sha2-512\r\ndebug3: send packet: type 50\r\ndebug3: receive packet: type 52\r\ndebug1: Enabling compression at level 6.\r\ndebug1: Authentication succeeded (publickey).\r\nAuthenticated to 192.168.1.6 ([192.168.1.6]:22).\r\ndebug1: setting up multiplex master socket\r\ndebug3: muxserver_listen: temporary control path /home/akokov/.ansible/cp/c3e3deaa98.ACLM3VoNbPYzr831\r\ndebug2: fd 4 setting O_NONBLOCK\r\ndebug3: fd 4 is O_NONBLOCK\r\ndebug3: fd 4 is O_NONBLOCK\r\ndebug1: channel 0: new [/home/akokov/.ansible/cp/c3e3deaa98]\r\ndebug3: muxserver_listen: mux listener channel 0 fd 4\r\ndebug2: fd 3 setting TCP_NODELAY\r\ndebug3: ssh_packet_set_tos: set IP_TOS 0x08\r\ndebug1: control_persist_detach: backgrounding master process\r\ndebug2: control_persist_detach: background process is 14503\r\ndebug2: fd 4 setting O_NONBLOCK\r\ndebug1: forking to background\r\ndebug1: Entering interactive session.\r\ndebug1: pledge: id\r\ndebug2: set_control_persist_exit_time: schedule exit in 60 seconds\r\ndebug1: multiplexing control connection\r\ndebug2: fd 5 setting O_NONBLOCK\r\ndebug3: fd 5 is O_NONBLOCK\r\ndebug1: channel 1: new [mux-control]\r\ndebug3: channel_post_mux_listener: new mux channel 1 fd 5\r\ndebug3: mux_master_read_cb: channel 1: hello sent\r\ndebug2: set_control_persist_exit_time: cancel scheduled exit\r\ndebug3: mux_master_read_cb: channel 1 packet type 0x00000001 len 4\r\ndebug2: mux_master_process_hello: channel 1 slave version 4\r\ndebug2: mux_client_hello_exchange: master version 4\r\ndebug3: mux_client_forwards: request forwardings: 0 local, 0 remote\r\ndebug3: mux_client_request_session: entering\r\ndebug3: mux_client_request_alive: entering\r\ndebug3: mux_master_read_cb: channel 1 packet type 0x10000004 len 4\r\ndebug2: mux_master_process_alive_check: channel 1: alive check\r\ndebug3: mux_client_request_alive: done pid = 14505\r\ndebug3: mux_master_read_cb: channel 1 packet type 0x10000002 len 344\r\ndebug3: mux_client_request_session: session request sent\r\ndebug2: mux_master_process_new_session: channel 1: request tty 0, X 0, agent 0, subsys 0, term "xterm-256color", cmd "/bin/sh -c \'echo ~root && sleep 0\'", env 10\r\ndebug3: mux_master_process_new_session: got fds stdin 6, stdout 7, stderr 8\r\ndebug2: fd 7 setting O_NONBLOCK\r\ndebug2: fd 8 setting O_NONBLOCK\r\ndebug1: channel 2: new [client-session]\r\ndebug2: mux_master_process_new_session: channel_new: 2 linked to control channel 1\r\ndebug2: channel 2: send open\r\ndebug3: send packet: type 90\r\ndebug3: receive packet: type 80\r\ndebug1: client_input_global_request: rtype hostkeys-00@openssh.com want_reply 0\r\ndebug3: receive packet: type 4\r\ndebug1: Remote: /root/.ssh/authorized_keys:1: key options: agent-forwarding port-forwarding pty user-rc x11-forwarding\r\ndebug3: receive packet: type 4\r\ndebug1: Remote: /root/.ssh/authorized_keys:1: key options: agent-forwarding port-forwarding pty user-rc x11-forwarding\r\ndebug3: receive packet: type 91\r\ndebug2: channel_input_open_confirmation: channel 2: callback start\r\ndebug2: client_session2_setup: id 2\r\ndebug1: Sending environment.\r\ndebug1: Sending env LC_ADDRESS = ru_RU.UTF-8\r\ndebug2: channel 2: request env confirm 0\r\ndebug3: send packet: type 98\r\ndebug1: Sending env LC_NAME = ru_RU.UTF-8\r\ndebug2: channel 2: request env confirm 0\r\ndebug3: send packet: type 98\r\ndebug1: Sending env LC_MONETARY = ru_RU.UTF-8\r\ndebug2: channel 2: request env confirm 0\r\ndebug3: send packet: type 98\r\ndebug1: Sending env LC_PAPER = ru_RU.UTF-8\r\ndebug2: channel 2: request env confirm 0\r\ndebug3: send packet: type 98\r\ndebug1: Sending env LANG = en_US.UTF-8\r\ndebug2: channel 2: request env confirm 0\r\ndebug3: send packet: type 98\r\ndebug1: Sending env LC_IDENTIFICATION = ru_RU.UTF-8\r\ndebug2: channel 2: request env confirm 0\r\ndebug3: send packet: type 98\r\ndebug1: Sending env LC_TELEPHONE = ru_RU.UTF-8\r\ndebug2: channel 2: request env confirm 0\r\ndebug3: send packet: type 98\r\ndebug1: Sending env LC_MEASUREMENT = ru_RU.UTF-8\r\ndebug2: channel 2: request env confirm 0\r\ndebug3: send packet: type 98\r\ndebug1: Sending env LC_TIME = ru_RU.UTF-8\r\ndebug2: channel 2: request env confirm 0\r\ndebug3: send packet: type 98\r\ndebug1: Sending env LC_NUMERIC = ru_RU.UTF-8\r\ndebug2: channel 2: request env confirm 0\r\ndebug3: send packet: type 98\r\ndebug1: Sending command: /bin/sh -c \'echo ~root && sleep 0\'\r\ndebug2: channel 2: request exec confirm 1\r\ndebug3: send packet: type 98\r\ndebug3: mux_session_confirm: sending success reply\r\ndebug2: channel_input_open_confirmation: channel 2: callback done\r\ndebug2: channel 2: open confirm rwindow 0 rmax 32768\r\ndebug2: channel 2: rcvd adjust 2097152\r\ndebug3: receive packet: type 99\r\ndebug2: channel_input_status_confirm: type 99 id 2\r\ndebug2: exec request accepted on channel 2\r\ndebug3: receive packet: type 98\r\ndebug1: client_input_channel_req: channel 2 rtype exit-status reply 0\r\ndebug3: mux_exit_message: channel 2: exit message, exitval 0\r\ndebug3: receive packet: type 98\r\ndebug1: client_input_channel_req: channel 2 rtype eow@openssh.com reply 0\r\ndebug2: channel 2: rcvd eow\r\ndebug2: channel 2: chan_shutdown_read (i0 o0 sock -1 wfd 6 efd 8 [write])\r\ndebug2: channel 2: input open -> closed\r\ndebug3: receive packet: type 96\r\ndebug2: channel 2: rcvd eof\r\ndebug2: channel 2: output open -> drain\r\ndebug2: channel 2: obuf empty\r\ndebug2: channel 2: chan_shutdown_write (i3 o1 sock -1 wfd 7 efd 8 [write])\r\ndebug2: channel 2: output drain -> closed\r\ndebug3: receive packet: type 97\r\ndebug2: channel 2: rcvd close\r\ndebug3: channel 2: will not send data after close\r\ndebug2: channel 2: send close\r\ndebug3: send packet: type 97\r\ndebug2: channel 2: is dead\r\ndebug2: channel 2: gc: notify user\r\ndebug3: mux_master_session_cleanup_cb: entering for channel 2\r\ndebug2: channel 1: rcvd close\r\ndebug2: channel 1: output open -> drain\r\ndebug2: channel 1: chan_shutdown_read (i0 o1 sock 5 wfd 5 efd -1 [closed])\r\ndebug2: channel 1: input open -> closed\r\ndebug2: channel 2: gc: user detached\r\ndebug2: channel 2: is dead\r\ndebug2: channel 2: garbage collecting\r\ndebug1: channel 2: free: client-session, nchannels 3\r\ndebug3: channel 2: status: The following connections are open:\r\n  #1 mux-control (t16 nr0 i3/0 o1/16 e[closed]/0 fd 5/5/-1 sock 5 cc -1)\r\n  #2 client-session (t4 r0 i3/0 o3/0 e[write]/0 fd -1/-1/8 sock -1 cc -1)\r\n\r\ndebug2: channel 1: obuf empty\r\ndebug2: channel 1: chan_shutdown_write (i3 o1 sock 5 wfd 5 efd -1 [closed])\r\ndebug2: channel 1: output drain -> closed\r\ndebug2: channel 1: is dead (local)\r\ndebug2: channel 1: gc: notify user\r\ndebug3: mux_master_control_cleanup_cb: entering for channel 1\r\ndebug2: channel 1: gc: user detached\r\ndebug2: channel 1: is dead (local)\r\ndebug2: channel 1: garbage collecting\r\ndebug1: channel 1: free: mux-control, nchannels 2\r\ndebug3: channel 1: status: The following connections are open:\r\n  #1 mux-control (t16 nr0 i3/0 o3/0 e[closed]/0 fd 5/5/-1 sock 5 cc -1)\r\n\r\ndebug3: mux_client_read_packet: read header failed: Broken pipe\r\ndebug2: set_control_persist_exit_time: schedule exit in 60 seconds\r\ndebug2: Received exit status from master 0\r\n')
2020-04-19 02:29:06,934 p=14491 u=akokov |  <192.168.1.6> ESTABLISH SSH CONNECTION FOR USER: root
2020-04-19 02:29:06,936 p=14491 u=akokov |  <192.168.1.6> SSH: EXEC ssh -vvv -C -o ControlMaster=auto -o ControlPersist=60s -o KbdInteractiveAuthentication=no -o PreferredAuthentications=gssapi-with-mic,gssapi-keyex,hostbased,publickey -o PasswordAuthentication=no -o 'User="root"' -o ConnectTimeout=10 -o ControlPath=/home/akokov/.ansible/cp/c3e3deaa98 192.168.1.6 '/bin/sh -c '"'"'( umask 77 && mkdir -p "` echo /root/.ansible/tmp/ansible-tmp-1587252545.5932105-240199050038266 `" && echo ansible-tmp-1587252545.5932105-240199050038266="` echo /root/.ansible/tmp/ansible-tmp-1587252545.5932105-240199050038266 `" ) && sleep 0'"'"''
2020-04-19 02:29:06,957 p=14491 u=akokov |  <192.168.1.6> (0, b'ansible-tmp-1587252545.5932105-240199050038266=/root/.ansible/tmp/ansible-tmp-1587252545.5932105-240199050038266\n', b'OpenSSH_8.0p1 Ubuntu-6build1, OpenSSL 1.1.1c  28 May 2019\r\ndebug1: Reading configuration data /etc/ssh/ssh_config\r\ndebug1: /etc/ssh/ssh_config line 19: Applying options for *\r\ndebug2: resolve_canonicalize: hostname 192.168.1.6 is address\r\ndebug1: auto-mux: Trying existing master\r\ndebug2: fd 3 setting O_NONBLOCK\r\ndebug2: mux_client_hello_exchange: master version 4\r\ndebug3: mux_client_forwards: request forwardings: 0 local, 0 remote\r\ndebug3: mux_client_request_session: entering\r\ndebug3: mux_client_request_alive: entering\r\ndebug3: mux_client_request_alive: done pid = 14505\r\ndebug3: mux_client_request_session: session request sent\r\ndebug3: mux_client_read_packet: read header failed: Broken pipe\r\ndebug2: Received exit status from master 0\r\n')
2020-04-19 02:29:07,142 p=14491 u=akokov |  <test> Attempting python interpreter discovery
2020-04-19 02:29:07,143 p=14491 u=akokov |  <192.168.1.6> ESTABLISH SSH CONNECTION FOR USER: root
2020-04-19 02:29:07,143 p=14491 u=akokov |  <192.168.1.6> SSH: EXEC ssh -vvv -C -o ControlMaster=auto -o ControlPersist=60s -o KbdInteractiveAuthentication=no -o PreferredAuthentications=gssapi-with-mic,gssapi-keyex,hostbased,publickey -o PasswordAuthentication=no -o 'User="root"' -o ConnectTimeout=10 -o ControlPath=/home/akokov/.ansible/cp/c3e3deaa98 192.168.1.6 '/bin/sh -c '"'"'echo PLATFORM; uname; echo FOUND; command -v '"'"'"'"'"'"'"'"'/usr/bin/python'"'"'"'"'"'"'"'"'; command -v '"'"'"'"'"'"'"'"'python3.7'"'"'"'"'"'"'"'"'; command -v '"'"'"'"'"'"'"'"'python3.6'"'"'"'"'"'"'"'"'; command -v '"'"'"'"'"'"'"'"'python3.5'"'"'"'"'"'"'"'"'; command -v '"'"'"'"'"'"'"'"'python2.7'"'"'"'"'"'"'"'"'; command -v '"'"'"'"'"'"'"'"'python2.6'"'"'"'"'"'"'"'"'; command -v '"'"'"'"'"'"'"'"'/usr/libexec/platform-python'"'"'"'"'"'"'"'"'; command -v '"'"'"'"'"'"'"'"'/usr/bin/python3'"'"'"'"'"'"'"'"'; command -v '"'"'"'"'"'"'"'"'python'"'"'"'"'"'"'"'"'; echo ENDFOUND && sleep 0'"'"''
2020-04-19 02:29:07,162 p=14491 u=akokov |  <192.168.1.6> (0, b'PLATFORM\nLinux\nFOUND\n/usr/libexec/platform-python\nENDFOUND\n', b'OpenSSH_8.0p1 Ubuntu-6build1, OpenSSL 1.1.1c  28 May 2019\r\ndebug1: Reading configuration data /etc/ssh/ssh_config\r\ndebug1: /etc/ssh/ssh_config line 19: Applying options for *\r\ndebug2: resolve_canonicalize: hostname 192.168.1.6 is address\r\ndebug1: auto-mux: Trying existing master\r\ndebug2: fd 3 setting O_NONBLOCK\r\ndebug2: mux_client_hello_exchange: master version 4\r\ndebug3: mux_client_forwards: request forwardings: 0 local, 0 remote\r\ndebug3: mux_client_request_session: entering\r\ndebug3: mux_client_request_alive: entering\r\ndebug3: mux_client_request_alive: done pid = 14505\r\ndebug3: mux_client_request_session: session request sent\r\ndebug3: mux_client_read_packet: read header failed: Broken pipe\r\ndebug2: Received exit status from master 0\r\n')
2020-04-19 02:29:07,163 p=14491 u=akokov |  <192.168.1.6> ESTABLISH SSH CONNECTION FOR USER: root
2020-04-19 02:29:07,163 p=14491 u=akokov |  <192.168.1.6> SSH: EXEC ssh -vvv -C -o ControlMaster=auto -o ControlPersist=60s -o KbdInteractiveAuthentication=no -o PreferredAuthentications=gssapi-with-mic,gssapi-keyex,hostbased,publickey -o PasswordAuthentication=no -o 'User="root"' -o ConnectTimeout=10 -o ControlPath=/home/akokov/.ansible/cp/c3e3deaa98 192.168.1.6 '/bin/sh -c '"'"'/usr/libexec/platform-python && sleep 0'"'"''
2020-04-19 02:29:07,213 p=14491 u=akokov |  <192.168.1.6> (0, b'{"platform_dist_result": ["centos", "8.1.1911", "Core"], "osrelease_content": "NAME=\\"CentOS Linux\\"\\nVERSION=\\"8 (Core)\\"\\nID=\\"centos\\"\\nID_LIKE=\\"rhel fedora\\"\\nVERSION_ID=\\"8\\"\\nPLATFORM_ID=\\"platform:el8\\"\\nPRETTY_NAME=\\"CentOS Linux 8 (Core)\\"\\nANSI_COLOR=\\"0;31\\"\\nCPE_NAME=\\"cpe:/o:centos:centos:8\\"\\nHOME_URL=\\"https://www.centos.org/\\"\\nBUG_REPORT_URL=\\"https://bugs.centos.org/\\"\\n\\nCENTOS_MANTISBT_PROJECT=\\"CentOS-8\\"\\nCENTOS_MANTISBT_PROJECT_VERSION=\\"8\\"\\nREDHAT_SUPPORT_PRODUCT=\\"centos\\"\\nREDHAT_SUPPORT_PRODUCT_VERSION=\\"8\\"\\n\\n"}\n', b'OpenSSH_8.0p1 Ubuntu-6build1, OpenSSL 1.1.1c  28 May 2019\r\ndebug1: Reading configuration data /etc/ssh/ssh_config\r\ndebug1: /etc/ssh/ssh_config line 19: Applying options for *\r\ndebug2: resolve_canonicalize: hostname 192.168.1.6 is address\r\ndebug1: auto-mux: Trying existing master\r\ndebug2: fd 3 setting O_NONBLOCK\r\ndebug2: mux_client_hello_exchange: master version 4\r\ndebug3: mux_client_forwards: request forwardings: 0 local, 0 remote\r\ndebug3: mux_client_request_session: entering\r\ndebug3: mux_client_request_alive: entering\r\ndebug3: mux_client_request_alive: done pid = 14505\r\ndebug3: mux_client_request_session: session request sent\r\ndebug3: mux_client_read_packet: read header failed: Broken pipe\r\ndebug2: Received exit status from master 0\r\n')
2020-04-19 02:29:07,214 p=14491 u=akokov |  Using module file /usr/lib/python3/dist-packages/ansible/modules/system/setup.py
2020-04-19 02:29:07,215 p=14491 u=akokov |  <192.168.1.6> PUT /home/akokov/.ansible/tmp/ansible-local-14491f0q6ghcl/tmp4okoz128 TO /root/.ansible/tmp/ansible-tmp-1587252545.5932105-240199050038266/AnsiballZ_setup.py
2020-04-19 02:29:07,215 p=14491 u=akokov |  <192.168.1.6> SSH: EXEC sftp -b - -vvv -C -o ControlMaster=auto -o ControlPersist=60s -o KbdInteractiveAuthentication=no -o PreferredAuthentications=gssapi-with-mic,gssapi-keyex,hostbased,publickey -o PasswordAuthentication=no -o 'User="root"' -o ConnectTimeout=10 -o ControlPath=/home/akokov/.ansible/cp/c3e3deaa98 '[192.168.1.6]'
2020-04-19 02:29:07,251 p=14491 u=akokov |  <192.168.1.6> (0, b'sftp> put /home/akokov/.ansible/tmp/ansible-local-14491f0q6ghcl/tmp4okoz128 /root/.ansible/tmp/ansible-tmp-1587252545.5932105-240199050038266/AnsiballZ_setup.py\n', b'OpenSSH_8.0p1 Ubuntu-6build1, OpenSSL 1.1.1c  28 May 2019\r\ndebug1: Reading configuration data /etc/ssh/ssh_config\r\ndebug1: /etc/ssh/ssh_config line 19: Applying options for *\r\ndebug2: resolve_canonicalize: hostname 192.168.1.6 is address\r\ndebug1: auto-mux: Trying existing master\r\ndebug2: fd 3 setting O_NONBLOCK\r\ndebug2: mux_client_hello_exchange: master version 4\r\ndebug3: mux_client_forwards: request forwardings: 0 local, 0 remote\r\ndebug3: mux_client_request_session: entering\r\ndebug3: mux_client_request_alive: entering\r\ndebug3: mux_client_request_alive: done pid = 14505\r\ndebug3: mux_client_request_session: session request sent\r\ndebug2: Remote version: 3\r\ndebug2: Server supports extension "posix-rename@openssh.com" revision 1\r\ndebug2: Server supports extension "statvfs@openssh.com" revision 2\r\ndebug2: Server supports extension "fstatvfs@openssh.com" revision 2\r\ndebug2: Server supports extension "hardlink@openssh.com" revision 1\r\ndebug2: Server supports extension "fsync@openssh.com" revision 1\r\ndebug2: Server supports extension "lsetstat@openssh.com" revision 1\r\ndebug3: Sent message fd 3 T:16 I:1\r\ndebug3: SSH_FXP_REALPATH . -> /root size 0\r\ndebug3: Looking up /home/akokov/.ansible/tmp/ansible-local-14491f0q6ghcl/tmp4okoz128\r\ndebug3: Sent message fd 3 T:17 I:2\r\ndebug3: Received stat reply T:101 I:2\r\ndebug1: Couldn\'t stat remote file: No such file or directory\r\ndebug3: Sent message SSH2_FXP_OPEN I:3 P:/root/.ansible/tmp/ansible-tmp-1587252545.5932105-240199050038266/AnsiballZ_setup.py\r\ndebug3: Sent message SSH2_FXP_WRITE I:4 O:0 S:32768\r\ndebug3: SSH2_FXP_STATUS 0\r\ndebug3: In write loop, ack for 4 32768 bytes at 0\r\ndebug3: Sent message SSH2_FXP_WRITE I:5 O:32768 S:32768\r\ndebug3: Sent message SSH2_FXP_WRITE I:6 O:65536 S:32768\r\ndebug3: Sent message SSH2_FXP_WRITE I:7 O:98304 S:32768\r\ndebug3: Sent message SSH2_FXP_WRITE I:8 O:131072 S:32768\r\ndebug3: Sent message SSH2_FXP_WRITE I:9 O:163840 S:32768\r\ndebug3: Sent message SSH2_FXP_WRITE I:10 O:196608 S:32768\r\ndebug3: Sent message SSH2_FXP_WRITE I:11 O:229376 S:23102\r\ndebug3: SSH2_FXP_STATUS 0\r\ndebug3: In write loop, ack for 5 32768 bytes at 32768\r\ndebug3: SSH2_FXP_STATUS 0\r\ndebug3: In write loop, ack for 6 32768 bytes at 65536\r\ndebug3: SSH2_FXP_STATUS 0\r\ndebug3: In write loop, ack for 7 32768 bytes at 98304\r\ndebug3: SSH2_FXP_STATUS 0\r\ndebug3: In write loop, ack for 8 32768 bytes at 131072\r\ndebug3: SSH2_FXP_STATUS 0\r\ndebug3: In write loop, ack for 9 32768 bytes at 163840\r\ndebug3: SSH2_FXP_STATUS 0\r\ndebug3: In write loop, ack for 10 32768 bytes at 196608\r\ndebug3: SSH2_FXP_STATUS 0\r\ndebug3: In write loop, ack for 11 23102 bytes at 229376\r\ndebug3: Sent message SSH2_FXP_CLOSE I:4\r\ndebug3: SSH2_FXP_STATUS 0\r\ndebug3: mux_client_read_packet: read header failed: Broken pipe\r\ndebug2: Received exit status from master 0\r\n')
2020-04-19 02:29:07,251 p=14491 u=akokov |  <192.168.1.6> ESTABLISH SSH CONNECTION FOR USER: root
2020-04-19 02:29:07,251 p=14491 u=akokov |  <192.168.1.6> SSH: EXEC ssh -vvv -C -o ControlMaster=auto -o ControlPersist=60s -o KbdInteractiveAuthentication=no -o PreferredAuthentications=gssapi-with-mic,gssapi-keyex,hostbased,publickey -o PasswordAuthentication=no -o 'User="root"' -o ConnectTimeout=10 -o ControlPath=/home/akokov/.ansible/cp/c3e3deaa98 192.168.1.6 '/bin/sh -c '"'"'chmod u+x /root/.ansible/tmp/ansible-tmp-1587252545.5932105-240199050038266/ /root/.ansible/tmp/ansible-tmp-1587252545.5932105-240199050038266/AnsiballZ_setup.py && sleep 0'"'"''
2020-04-19 02:29:07,271 p=14491 u=akokov |  <192.168.1.6> (0, b'', b'OpenSSH_8.0p1 Ubuntu-6build1, OpenSSL 1.1.1c  28 May 2019\r\ndebug1: Reading configuration data /etc/ssh/ssh_config\r\ndebug1: /etc/ssh/ssh_config line 19: Applying options for *\r\ndebug2: resolve_canonicalize: hostname 192.168.1.6 is address\r\ndebug1: auto-mux: Trying existing master\r\ndebug2: fd 3 setting O_NONBLOCK\r\ndebug2: mux_client_hello_exchange: master version 4\r\ndebug3: mux_client_forwards: request forwardings: 0 local, 0 remote\r\ndebug3: mux_client_request_session: entering\r\ndebug3: mux_client_request_alive: entering\r\ndebug3: mux_client_request_alive: done pid = 14505\r\ndebug3: mux_client_request_session: session request sent\r\ndebug3: mux_client_read_packet: read header failed: Broken pipe\r\ndebug2: Received exit status from master 0\r\n')
2020-04-19 02:29:07,271 p=14491 u=akokov |  <192.168.1.6> ESTABLISH SSH CONNECTION FOR USER: root
2020-04-19 02:29:07,272 p=14491 u=akokov |  <192.168.1.6> SSH: EXEC ssh -vvv -C -o ControlMaster=auto -o ControlPersist=60s -o KbdInteractiveAuthentication=no -o PreferredAuthentications=gssapi-with-mic,gssapi-keyex,hostbased,publickey -o PasswordAuthentication=no -o 'User="root"' -o ConnectTimeout=10 -o ControlPath=/home/akokov/.ansible/cp/c3e3deaa98 -tt 192.168.1.6 '/bin/sh -c '"'"'/usr/libexec/platform-python /root/.ansible/tmp/ansible-tmp-1587252545.5932105-240199050038266/AnsiballZ_setup.py && sleep 0'"'"''
2020-04-19 02:29:07,787 p=14491 u=akokov |  <192.168.1.6> (0, b'\r\n{"ansible_facts": {"ansible_distribution": "CentOS", "ansible_distribution_release": "Core", "ansible_distribution_version": "8.1", "ansible_distribution_major_version": "8", "ansible_distribution_file_path": "/etc/redhat-release", "ansible_distribution_file_variety": "RedHat", "ansible_distribution_file_parsed": true, "ansible_os_family": "RedHat", "ansible_system": "Linux", "ansible_kernel": "4.18.0-147.8.1.el8_1.x86_64", "ansible_machine": "x86_64", "ansible_python_version": "3.6.8", "ansible_fqdn": "192.168.1.6", "ansible_hostname": "192", "ansible_nodename": "192.168.1.6", "ansible_domain": "168.1.6", "ansible_userspace_bits": "64", "ansible_architecture": "x86_64", "ansible_userspace_architecture": "x86_64", "ansible_machine_id": "fea6ec107fcd4c12955cbcd0194b5baa", "ansible_virtualization_role": "guest", "ansible_virtualization_type": "virtualbox", "ansible_date_time": {"year": "2020", "month": "04", "weekday": "Saturday", "weekday_number": "6", "weeknumber": "15", "day": "18", "hour": "19", "minute": "29", "second": "07", "epoch": "1587252547", "date": "2020-04-18", "time": "19:29:07", "iso8601_micro": "2020-04-18T23:29:07.523195Z", "iso8601": "2020-04-18T23:29:07Z", "iso8601_basic": "20200418T192907523132", "iso8601_basic_short": "20200418T192907", "tz": "EDT", "tz_offset": "-0400"}, "ansible_ssh_host_key_rsa_public": "AAAAB3NzaC1yc2EAAAADAQABAAABgQC3r6TY2Vev50hmcuRkV2OnPVYDlf4vz7zpU+RbLeDwIQyc4731BQwzc0dPDs1ABIS/R9wu6xl8ePrlt7kd6Ww4vFTSa/GOVahykx9gUBhfrcTcJVWyoRlY+R1VHD9UZXTUKCKsixNNGsX0Q+XAIV3Lpt/of6gPeM7x6+6oqsr3tbbAwc5edFoJ5JqsMSNm10q/2ULt0ZxzgUM3vBKcLUQvD0T+KElAPOAth/l/OmyS/30lP8rNKtsBOOVYxox9rVlKJYwIVB528WzmXAeJgUSSDr2zoekLdU2d+ffzrZ03xuFbgGtb7SfPpFJsarTwcRguiUNiQGTVcJtwTDIkedujPUgODQRK5QeVjh5UIIJLU/D5sivMIj1CmHx5uGIBaFitpV5qiUwMt21hQirYq9p7POdM5e0dptuHCtgMyFsuj9fdnqHYqDtv0vK64WkYfhJw2vX6VrB83Rw+TgEMjA0l9e5SL3IRDlAoDepIP+dUP/Iyhr6kjJTyo6zkmyXDrC8=", "ansible_ssh_host_key_ecdsa_public": "AAAAE2VjZHNhLXNoYTItbmlzdHAyNTYAAAAIbmlzdHAyNTYAAABBBPSut19lix/qwhSbDKuU3Qq9CQtRBI7SyUBL7gsXg4UqhNgMvezo0aQyKgKwxVVYS36gSf5fPUkW1jm+DMcRRA4=", "ansible_ssh_host_key_ed25519_public": "AAAAC3NzaC1lZDI1NTE5AAAAIGZwe7W9tdRybSED7YXGFVOyDZkt3VMSbeV7gzplu0sl", "ansible_cmdline": {"BOOT_IMAGE": "(hd0,msdos1)/vmlinuz-4.18.0-147.8.1.el8_1.x86_64", "root": "/dev/mapper/cl_devops-root", "ro": true, "crashkernel": "auto", "resume": "/dev/mapper/cl_devops-swap", "rd.lvm.lv": "cl_devops/swap", "rhgb": true, "quiet": true}, "ansible_proc_cmdline": {"BOOT_IMAGE": "(hd0,msdos1)/vmlinuz-4.18.0-147.8.1.el8_1.x86_64", "root": "/dev/mapper/cl_devops-root", "ro": true, "crashkernel": "auto", "resume": "/dev/mapper/cl_devops-swap", "rd.lvm.lv": ["cl_devops/root", "cl_devops/swap"], "rhgb": true, "quiet": true}, "ansible_user_id": "root", "ansible_user_uid": 0, "ansible_user_gid": 0, "ansible_user_gecos": "root", "ansible_user_dir": "/root", "ansible_user_shell": "/bin/bash", "ansible_real_user_id": 0, "ansible_effective_user_id": 0, "ansible_real_group_id": 0, "ansible_effective_group_id": 0, "ansible_python": {"version": {"major": 3, "minor": 6, "micro": 8, "releaselevel": "final", "serial": 0}, "version_info": [3, 6, 8, "final", 0], "executable": "/usr/libexec/platform-python", "has_sslcontext": true, "type": "cpython"}, "ansible_processor": ["0", "GenuineIntel", "Intel(R) Core(TM) i5-8300H CPU @ 2.30GHz"], "ansible_processor_count": 1, "ansible_processor_cores": 1, "ansible_processor_threads_per_core": 1, "ansible_processor_vcpus": 1, "ansible_memtotal_mb": 1301, "ansible_memfree_mb": 712, "ansible_swaptotal_mb": 819, "ansible_swapfree_mb": 819, "ansible_memory_mb": {"real": {"total": 1301, "used": 589, "free": 712}, "nocache": {"free": 1052, "used": 249}, "swap": {"total": 819, "free": 819, "used": 0, "cached": 0}}, "ansible_bios_date": "12/01/2006", "ansible_bios_version": "VirtualBox", "ansible_form_factor": "Other", "ansible_product_name": "VirtualBox", "ansible_product_serial": "0", "ansible_product_uuid": "6dd321a7-9817-674d-9480-05afb3e88f88", "ansible_product_version": "1.2", "ansible_system_vendor": "innotek GmbH", "ansible_devices": {"dm-1": {"virtual": 1, "links": {"ids": ["dm-name-cl_devops-swap", "dm-uuid-LVM-Qns1pBp6uVBow6q5yWbWH4ID2PtF7yVXH2h9qNIAKVOCjBTYF2ddZiDTgFcKA581"], "uuids": ["4e2945b6-d6ea-4413-8c12-d968ae4f033b"], "labels": [], "masters": []}, "vendor": null, "model": null, "sas_address": null, "sas_device_handle": null, "serial": "VB88b2aca2", "removable": "0", "support_discard": "0", "partitions": {}, "rotational": "1", "scheduler_mode": "", "sectors": "1679360", "sectorsize": "512", "size": "820.00 MB", "host": "", "holders": []}, "dm-0": {"virtual": 1, "links": {"ids": ["dm-name-cl_devops-root", "dm-uuid-LVM-Qns1pBp6uVBow6q5yWbWH4ID2PtF7yVXsNDRO5CnhRYxxOKH7hmo3nw6SBgY94sz"], "uuids": ["6645efc3-e130-496c-8b43-bb2de84d148b"], "labels": [], "masters": []}, "vendor": null, "model": null, "sas_address": null, "sas_device_handle": null, "serial": "VB88b2aca2", "removable": "0", "support_discard": "0", "partitions": {}, "rotational": "1", "scheduler_mode": "", "sectors": "12992512", "sectorsize": "512", "size": "6.20 GB", "host": "", "holders": []}, "sda": {"virtual": 1, "links": {"ids": ["ata-VBOX_HARDDISK_VB88b2aca2-fa5d82d4"], "uuids": [], "labels": [], "masters": []}, "vendor": "ATA", "model": "VBOX HARDDISK", "sas_address": null, "sas_device_handle": null, "serial": "VB88b2aca2", "removable": "0", "support_discard": "0", "partitions": {"sda2": {"links": {"ids": ["ata-VBOX_HARDDISK_VB88b2aca2-fa5d82d4-part2", "lvm-pv-uuid-NgqnZc-uqHB-punk-62HL-Jcb6-afEt-VBdhp9"], "uuids": [], "labels": [], "masters": ["dm-0", "dm-1"]}, "start": "2099200", "sectors": "14678016", "sectorsize": 512, "size": "7.00 GB", "uuid": null, "holders": ["cl_devops-swap", "cl_devops-root"]}, "sda1": {"links": {"ids": ["ata-VBOX_HARDDISK_VB88b2aca2-fa5d82d4-part1"], "uuids": ["8ab467f0-9f8a-43b7-82cc-1705457dc878"], "labels": [], "masters": []}, "start": "2048", "sectors": "2097152", "sectorsize": 512, "size": "1.00 GB", "uuid": "8ab467f0-9f8a-43b7-82cc-1705457dc878", "holders": []}}, "rotational": "1", "scheduler_mode": "mq-deadline", "sectors": "16777216", "sectorsize": "512", "size": "8.00 GB", "host": "SATA controller: Intel Corporation 82801HM/HEM (ICH8M/ICH8M-E) SATA Controller [AHCI mode] (rev 02)", "holders": []}}, "ansible_device_links": {"ids": {"dm-1": ["dm-name-cl_devops-swap", "dm-uuid-LVM-Qns1pBp6uVBow6q5yWbWH4ID2PtF7yVXH2h9qNIAKVOCjBTYF2ddZiDTgFcKA581"], "dm-0": ["dm-name-cl_devops-root", "dm-uuid-LVM-Qns1pBp6uVBow6q5yWbWH4ID2PtF7yVXsNDRO5CnhRYxxOKH7hmo3nw6SBgY94sz"], "sda1": ["ata-VBOX_HARDDISK_VB88b2aca2-fa5d82d4-part1"], "sda2": ["ata-VBOX_HARDDISK_VB88b2aca2-fa5d82d4-part2", "lvm-pv-uuid-NgqnZc-uqHB-punk-62HL-Jcb6-afEt-VBdhp9"], "sda": ["ata-VBOX_HARDDISK_VB88b2aca2-fa5d82d4"]}, "uuids": {"dm-1": ["4e2945b6-d6ea-4413-8c12-d968ae4f033b"], "dm-0": ["6645efc3-e130-496c-8b43-bb2de84d148b"], "sda1": ["8ab467f0-9f8a-43b7-82cc-1705457dc878"]}, "labels": {}, "masters": {"sda2": ["dm-0", "dm-1"]}}, "ansible_uptime_seconds": 4382, "ansible_lvm": {"lvs": {"root": {"size_g": "6.20", "vg": "cl_devops"}, "swap": {"size_g": "0.80", "vg": "cl_devops"}}, "vgs": {"cl_devops": {"size_g": "7.00", "free_g": "0", "num_lvs": "2", "num_pvs": "1"}}, "pvs": {"/dev/sda2": {"size_g": "7.00", "free_g": "0", "vg": "cl_devops"}}}, "ansible_mounts": [{"mount": "/", "device": "/dev/mapper/cl_devops-root", "fstype": "xfs", "options": "rw,seclabel,relatime,attr2,inode64,noquota", "size_total": 6641680384, "size_available": 4753858560, "block_size": 4096, "block_total": 1621504, "block_available": 1160610, "block_used": 460894, "inode_total": 3248128, "inode_available": 3202327, "inode_used": 45801, "uuid": "6645efc3-e130-496c-8b43-bb2de84d148b"}, {"mount": "/boot", "device": "/dev/sda1", "fstype": "ext4", "options": "rw,seclabel,relatime", "size_total": 1023303680, "size_available": 806760448, "block_size": 4096, "block_total": 249830, "block_available": 196963, "block_used": 52867, "inode_total": 65536, "inode_available": 65227, "inode_used": 309, "uuid": "8ab467f0-9f8a-43b7-82cc-1705457dc878"}], "ansible_system_capabilities_enforced": "True", "ansible_system_capabilities": ["cap_chown", "cap_dac_override", "cap_dac_read_search", "cap_fowner", "cap_fsetid", "cap_kill", "cap_setgid", "cap_setuid", "cap_setpcap", "cap_linux_immutable", "cap_net_bind_service", "cap_net_broadcast", "cap_net_admin", "cap_net_raw", "cap_ipc_lock", "cap_ipc_owner", "cap_sys_module", "cap_sys_rawio", "cap_sys_chroot", "cap_sys_ptrace", "cap_sys_pacct", "cap_sys_admin", "cap_sys_boot", "cap_sys_nice", "cap_sys_resource", "cap_sys_time", "cap_sys_tty_config", "cap_mknod", "cap_lease", "cap_audit_write", "cap_audit_control", "cap_setfcap", "cap_mac_override", "cap_mac_admin", "cap_syslog", "cap_wake_alarm", "cap_block_suspend", "cap_audit_read+ep"], "ansible_selinux_python_present": true, "ansible_selinux": {"status": "enabled", "policyvers": 31, "config_mode": "enforcing", "mode": "enforcing", "type": "targeted"}, "ansible_dns": {"nameservers": ["fe80::1%enp0s3", "192.168.1.1"]}, "ansible_iscsi_iqn": "iqn.1994-05.com.redhat:15c42d8a88fd", "ansible_is_chroot": false, "ansible_fibre_channel_wwn": [], "ansible_lsb": {}, "ansible_local": {}, "ansible_env": {"LS_COLORS": "rs=0:di=38;5;33:ln=38;5;51:mh=00:pi=40;38;5;11:so=38;5;13:do=38;5;5:bd=48;5;232;38;5;11:cd=48;5;232;38;5;3:or=48;5;232;38;5;9:mi=01;05;37;41:su=48;5;196;38;5;15:sg=48;5;11;38;5;16:ca=48;5;196;38;5;226:tw=48;5;10;38;5;16:ow=48;5;10;38;5;21:st=48;5;21;38;5;15:ex=38;5;40:*.tar=38;5;9:*.tgz=38;5;9:*.arc=38;5;9:*.arj=38;5;9:*.taz=38;5;9:*.lha=38;5;9:*.lz4=38;5;9:*.lzh=38;5;9:*.lzma=38;5;9:*.tlz=38;5;9:*.txz=38;5;9:*.tzo=38;5;9:*.t7z=38;5;9:*.zip=38;5;9:*.z=38;5;9:*.dz=38;5;9:*.gz=38;5;9:*.lrz=38;5;9:*.lz=38;5;9:*.lzo=38;5;9:*.xz=38;5;9:*.zst=38;5;9:*.tzst=38;5;9:*.bz2=38;5;9:*.bz=38;5;9:*.tbz=38;5;9:*.tbz2=38;5;9:*.tz=38;5;9:*.deb=38;5;9:*.rpm=38;5;9:*.jar=38;5;9:*.war=38;5;9:*.ear=38;5;9:*.sar=38;5;9:*.rar=38;5;9:*.alz=38;5;9:*.ace=38;5;9:*.zoo=38;5;9:*.cpio=38;5;9:*.7z=38;5;9:*.rz=38;5;9:*.cab=38;5;9:*.wim=38;5;9:*.swm=38;5;9:*.dwm=38;5;9:*.esd=38;5;9:*.jpg=38;5;13:*.jpeg=38;5;13:*.mjpg=38;5;13:*.mjpeg=38;5;13:*.gif=38;5;13:*.bmp=38;5;13:*.pbm=38;5;13:*.pgm=38;5;13:*.ppm=38;5;13:*.tga=38;5;13:*.xbm=38;5;13:*.xpm=38;5;13:*.tif=38;5;13:*.tiff=38;5;13:*.png=38;5;13:*.svg=38;5;13:*.svgz=38;5;13:*.mng=38;5;13:*.pcx=38;5;13:*.mov=38;5;13:*.mpg=38;5;13:*.mpeg=38;5;13:*.m2v=38;5;13:*.mkv=38;5;13:*.webm=38;5;13:*.ogm=38;5;13:*.mp4=38;5;13:*.m4v=38;5;13:*.mp4v=38;5;13:*.vob=38;5;13:*.qt=38;5;13:*.nuv=38;5;13:*.wmv=38;5;13:*.asf=38;5;13:*.rm=38;5;13:*.rmvb=38;5;13:*.flc=38;5;13:*.avi=38;5;13:*.fli=38;5;13:*.flv=38;5;13:*.gl=38;5;13:*.dl=38;5;13:*.xcf=38;5;13:*.xwd=38;5;13:*.yuv=38;5;13:*.cgm=38;5;13:*.emf=38;5;13:*.ogv=38;5;13:*.ogx=38;5;13:*.aac=38;5;45:*.au=38;5;45:*.flac=38;5;45:*.m4a=38;5;45:*.mid=38;5;45:*.midi=38;5;45:*.mka=38;5;45:*.mp3=38;5;45:*.mpc=38;5;45:*.ogg=38;5;45:*.ra=38;5;45:*.wav=38;5;45:*.oga=38;5;45:*.opus=38;5;45:*.spx=38;5;45:*.xspf=38;5;45:", "LC_MEASUREMENT": "ru_RU.UTF-8", "SSH_CONNECTION": "192.168.1.2 54196 192.168.1.6 22", "LC_PAPER": "ru_RU.UTF-8", "LC_MONETARY": "ru_RU.UTF-8", "_": "/usr/libexec/platform-python", "LANG": "C", "LC_NAME": "ru_RU.UTF-8", "XDG_SESSION_ID": "9", "USER": "root", "SELINUX_ROLE_REQUESTED": "", "PWD": "/root", "HOME": "/root", "SSH_CLIENT": "192.168.1.2 54196 22", "SELINUX_LEVEL_REQUESTED": "", "LC_ADDRESS": "ru_RU.UTF-8", "LC_NUMERIC": "ru_RU.UTF-8", "SSH_TTY": "/dev/pts/0", "SHELL": "/bin/bash", "TERM": "xterm-256color", "SELINUX_USE_CURRENT_RANGE": "", "SHLVL": "2", "LC_TELEPHONE": "ru_RU.UTF-8", "LOGNAME": "root", "DBUS_SESSION_BUS_ADDRESS": "unix:path=/run/user/0/bus", "XDG_RUNTIME_DIR": "/run/user/0", "PATH": "/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin", "LC_IDENTIFICATION": "ru_RU.UTF-8", "LESSOPEN": "||/usr/bin/lesspipe.sh %s", "LC_TIME": "ru_RU.UTF-8", "LC_CTYPE": "C.UTF-8", "LC_ALL": "C", "LC_MESSAGES": "C"}, "ansible_fips": false, "ansible_hostnqn": "", "ansible_apparmor": {"status": "disabled"}, "ansible_pkg_mgr": "dnf", "ansible_interfaces": ["enp0s3", "lo"], "ansible_lo": {"device": "lo", "mtu": 65536, "active": true, "type": "loopback", "promisc": false, "ipv4": {"address": "127.0.0.1", "broadcast": "host", "netmask": "255.0.0.0", "network": "127.0.0.0"}, "ipv6": [{"address": "::1", "prefix": "128", "scope": "host"}], "features": {"rx_checksumming": "on [fixed]", "tx_checksumming": "on", "tx_checksum_ipv4": "off [fixed]", "tx_checksum_ip_generic": "on [fixed]", "tx_checksum_ipv6": "off [fixed]", "tx_checksum_fcoe_crc": "off [fixed]", "tx_checksum_sctp": "on [fixed]", "scatter_gather": "on", "tx_scatter_gather": "on [fixed]", "tx_scatter_gather_fraglist": "on [fixed]", "tcp_segmentation_offload": "on", "tx_tcp_segmentation": "on", "tx_tcp_ecn_segmentation": "on", "tx_tcp_mangleid_segmentation": "on", "tx_tcp6_segmentation": "on", "generic_segmentation_offload": "on", "generic_receive_offload": "on", "large_receive_offload": "off [fixed]", "rx_vlan_offload": "off [fixed]", "tx_vlan_offload": "off [fixed]", "ntuple_filters": "off [fixed]", "receive_hashing": "off [fixed]", "highdma": "on [fixed]", "rx_vlan_filter": "off [fixed]", "vlan_challenged": "on [fixed]", "tx_lockless": "on [fixed]", "netns_local": "on [fixed]", "tx_gso_robust": "off [fixed]", "tx_fcoe_segmentation": "off [fixed]", "tx_gre_segmentation": "off [fixed]", "tx_gre_csum_segmentation": "off [fixed]", "tx_ipxip4_segmentation": "off [fixed]", "tx_ipxip6_segmentation": "off [fixed]", "tx_udp_tnl_segmentation": "off [fixed]", "tx_udp_tnl_csum_segmentation": "off [fixed]", "tx_gso_partial": "off [fixed]", "tx_sctp_segmentation": "on", "tx_esp_segmentation": "off [fixed]", "tx_udp_segmentation": "off [fixed]", "tls_hw_rx_offload": "off [fixed]", "fcoe_mtu": "off [fixed]", "tx_nocache_copy": "off [fixed]", "loopback": "on [fixed]", "rx_fcs": "off [fixed]", "rx_all": "off [fixed]", "tx_vlan_stag_hw_insert": "off [fixed]", "rx_vlan_stag_hw_parse": "off [fixed]", "rx_vlan_stag_filter": "off [fixed]", "l2_fwd_offload": "off [fixed]", "hw_tc_offload": "off [fixed]", "esp_hw_offload": "off [fixed]", "esp_tx_csum_hw_offload": "off [fixed]", "rx_udp_tunnel_port_offload": "off [fixed]", "tls_hw_tx_offload": "off [fixed]", "rx_gro_hw": "off [fixed]", "tls_hw_record": "off [fixed]"}, "timestamping": ["tx_software", "rx_software", "software"], "hw_timestamp_filters": []}, "ansible_enp0s3": {"device": "enp0s3", "macaddress": "08:00:27:1c:77:48", "mtu": 1500, "active": true, "module": "e1000", "type": "ether", "pciid": "0000:00:03.0", "speed": 1000, "promisc": false, "ipv4": {"address": "192.168.1.6", "broadcast": "192.168.1.255", "netmask": "255.255.255.0", "network": "192.168.1.0"}, "ipv6": [{"address": "2a00:1370:811f:897a:4b5c:d8e4:259c:34a5", "prefix": "64", "scope": "global"}, {"address": "fe80::b830:78dd:25c1:fc2a", "prefix": "64", "scope": "link"}], "features": {"rx_checksumming": "off", "tx_checksumming": "on", "tx_checksum_ipv4": "off [fixed]", "tx_checksum_ip_generic": "on", "tx_checksum_ipv6": "off [fixed]", "tx_checksum_fcoe_crc": "off [fixed]", "tx_checksum_sctp": "off [fixed]", "scatter_gather": "on", "tx_scatter_gather": "on", "tx_scatter_gather_fraglist": "off [fixed]", "tcp_segmentation_offload": "on", "tx_tcp_segmentation": "on", "tx_tcp_ecn_segmentation": "off [fixed]", "tx_tcp_mangleid_segmentation": "off", "tx_tcp6_segmentation": "off [fixed]", "generic_segmentation_offload": "on", "generic_receive_offload": "on", "large_receive_offload": "off [fixed]", "rx_vlan_offload": "on", "tx_vlan_offload": "on [fixed]", "ntuple_filters": "off [fixed]", "receive_hashing": "off [fixed]", "highdma": "off [fixed]", "rx_vlan_filter": "on [fixed]", "vlan_challenged": "off [fixed]", "tx_lockless": "off [fixed]", "netns_local": "off [fixed]", "tx_gso_robust": "off [fixed]", "tx_fcoe_segmentation": "off [fixed]", "tx_gre_segmentation": "off [fixed]", "tx_gre_csum_segmentation": "off [fixed]", "tx_ipxip4_segmentation": "off [fixed]", "tx_ipxip6_segmentation": "off [fixed]", "tx_udp_tnl_segmentation": "off [fixed]", "tx_udp_tnl_csum_segmentation": "off [fixed]", "tx_gso_partial": "off [fixed]", "tx_sctp_segmentation": "off [fixed]", "tx_esp_segmentation": "off [fixed]", "tx_udp_segmentation": "off [fixed]", "tls_hw_rx_offload": "off [fixed]", "fcoe_mtu": "off [fixed]", "tx_nocache_copy": "off", "loopback": "off [fixed]", "rx_fcs": "off", "rx_all": "off", "tx_vlan_stag_hw_insert": "off [fixed]", "rx_vlan_stag_hw_parse": "off [fixed]", "rx_vlan_stag_filter": "off [fixed]", "l2_fwd_offload": "off [fixed]", "hw_tc_offload": "off [fixed]", "esp_hw_offload": "off [fixed]", "esp_tx_csum_hw_offload": "off [fixed]", "rx_udp_tunnel_port_offload": "off [fixed]", "tls_hw_tx_offload": "off [fixed]", "rx_gro_hw": "off [fixed]", "tls_hw_record": "off [fixed]"}, "timestamping": ["tx_software", "rx_software", "software"], "hw_timestamp_filters": []}, "ansible_default_ipv4": {"gateway": "192.168.1.1", "interface": "enp0s3", "address": "192.168.1.6", "broadcast": "192.168.1.255", "netmask": "255.255.255.0", "network": "192.168.1.0", "macaddress": "08:00:27:1c:77:48", "mtu": 1500, "type": "ether", "alias": "enp0s3"}, "ansible_default_ipv6": {"gateway": "fe80::1", "interface": "enp0s3", "address": "2a00:1370:811f:897a:4b5c:d8e4:259c:34a5", "prefix": "64", "scope": "global", "macaddress": "08:00:27:1c:77:48", "mtu": 1500, "type": "ether"}, "ansible_all_ipv4_addresses": ["192.168.1.6"], "ansible_all_ipv6_addresses": ["2a00:1370:811f:897a:4b5c:d8e4:259c:34a5", "fe80::b830:78dd:25c1:fc2a"], "ansible_service_mgr": "systemd", "gather_subset": ["all"], "module_setup": true}, "invocation": {"module_args": {"gather_subset": ["all"], "gather_timeout": 10, "filter": "*", "fact_path": "/etc/ansible/facts.d"}}}\r\n', b'OpenSSH_8.0p1 Ubuntu-6build1, OpenSSL 1.1.1c  28 May 2019\r\ndebug1: Reading configuration data /etc/ssh/ssh_config\r\ndebug1: /etc/ssh/ssh_config line 19: Applying options for *\r\ndebug2: resolve_canonicalize: hostname 192.168.1.6 is address\r\ndebug1: auto-mux: Trying existing master\r\ndebug2: fd 3 setting O_NONBLOCK\r\ndebug2: mux_client_hello_exchange: master version 4\r\ndebug3: mux_client_forwards: request forwardings: 0 local, 0 remote\r\ndebug3: mux_client_request_session: entering\r\ndebug3: mux_client_request_alive: entering\r\ndebug3: mux_client_request_alive: done pid = 14505\r\ndebug3: mux_client_request_session: session request sent\r\ndebug3: mux_client_read_packet: read header failed: Broken pipe\r\ndebug2: Received exit status from master 0\r\nShared connection to 192.168.1.6 closed.\r\n')
2020-04-19 02:29:07,789 p=14491 u=akokov |  <192.168.1.6> ESTABLISH SSH CONNECTION FOR USER: root
2020-04-19 02:29:07,790 p=14491 u=akokov |  <192.168.1.6> SSH: EXEC ssh -vvv -C -o ControlMaster=auto -o ControlPersist=60s -o KbdInteractiveAuthentication=no -o PreferredAuthentications=gssapi-with-mic,gssapi-keyex,hostbased,publickey -o PasswordAuthentication=no -o 'User="root"' -o ConnectTimeout=10 -o ControlPath=/home/akokov/.ansible/cp/c3e3deaa98 192.168.1.6 '/bin/sh -c '"'"'rm -f -r /root/.ansible/tmp/ansible-tmp-1587252545.5932105-240199050038266/ > /dev/null 2>&1 && sleep 0'"'"''
2020-04-19 02:29:07,808 p=14491 u=akokov |  <192.168.1.6> (0, b'', b'OpenSSH_8.0p1 Ubuntu-6build1, OpenSSL 1.1.1c  28 May 2019\r\ndebug1: Reading configuration data /etc/ssh/ssh_config\r\ndebug1: /etc/ssh/ssh_config line 19: Applying options for *\r\ndebug2: resolve_canonicalize: hostname 192.168.1.6 is address\r\ndebug1: auto-mux: Trying existing master\r\ndebug2: fd 3 setting O_NONBLOCK\r\ndebug2: mux_client_hello_exchange: master version 4\r\ndebug3: mux_client_forwards: request forwardings: 0 local, 0 remote\r\ndebug3: mux_client_request_session: entering\r\ndebug3: mux_client_request_alive: entering\r\ndebug3: mux_client_request_alive: done pid = 14505\r\ndebug3: mux_client_request_session: session request sent\r\ndebug3: mux_client_read_packet: read header failed: Broken pipe\r\ndebug2: Received exit status from master 0\r\n')
2020-04-19 02:29:07,819 p=14491 u=akokov |  ok: [test]
2020-04-19 02:29:07,819 p=14491 u=akokov |  META: ran handlers
2020-04-19 02:29:07,829 p=14491 u=akokov |  TASK [Download Java JRE] ********************************************************************************************************************************************************************************************
2020-04-19 02:29:07,829 p=14491 u=akokov |  task path: /home/ansible/playbook-install.yml:11
2020-04-19 02:29:07,836 p=14491 u=akokov |  <192.168.1.6> ESTABLISH SSH CONNECTION FOR USER: root
2020-04-19 02:29:07,837 p=14491 u=akokov |  <192.168.1.6> SSH: EXEC ssh -vvv -C -o ControlMaster=auto -o ControlPersist=60s -o KbdInteractiveAuthentication=no -o PreferredAuthentications=gssapi-with-mic,gssapi-keyex,hostbased,publickey -o PasswordAuthentication=no -o 'User="root"' -o ConnectTimeout=10 -o ControlPath=/home/akokov/.ansible/cp/c3e3deaa98 192.168.1.6 '/bin/sh -c '"'"'echo ~root && sleep 0'"'"''
2020-04-19 02:29:07,857 p=14491 u=akokov |  <192.168.1.6> (0, b'/root\n', b'OpenSSH_8.0p1 Ubuntu-6build1, OpenSSL 1.1.1c  28 May 2019\r\ndebug1: Reading configuration data /etc/ssh/ssh_config\r\ndebug1: /etc/ssh/ssh_config line 19: Applying options for *\r\ndebug2: resolve_canonicalize: hostname 192.168.1.6 is address\r\ndebug1: auto-mux: Trying existing master\r\ndebug2: fd 3 setting O_NONBLOCK\r\ndebug2: mux_client_hello_exchange: master version 4\r\ndebug3: mux_client_forwards: request forwardings: 0 local, 0 remote\r\ndebug3: mux_client_request_session: entering\r\ndebug3: mux_client_request_alive: entering\r\ndebug3: mux_client_request_alive: done pid = 14505\r\ndebug3: mux_client_request_session: session request sent\r\ndebug3: mux_client_read_packet: read header failed: Broken pipe\r\ndebug2: Received exit status from master 0\r\n')
2020-04-19 02:29:07,857 p=14491 u=akokov |  <192.168.1.6> ESTABLISH SSH CONNECTION FOR USER: root
2020-04-19 02:29:07,857 p=14491 u=akokov |  <192.168.1.6> SSH: EXEC ssh -vvv -C -o ControlMaster=auto -o ControlPersist=60s -o KbdInteractiveAuthentication=no -o PreferredAuthentications=gssapi-with-mic,gssapi-keyex,hostbased,publickey -o PasswordAuthentication=no -o 'User="root"' -o ConnectTimeout=10 -o ControlPath=/home/akokov/.ansible/cp/c3e3deaa98 192.168.1.6 '/bin/sh -c '"'"'( umask 77 && mkdir -p "` echo /root/.ansible/tmp/ansible-tmp-1587252547.8363883-245698651101224 `" && echo ansible-tmp-1587252547.8363883-245698651101224="` echo /root/.ansible/tmp/ansible-tmp-1587252547.8363883-245698651101224 `" ) && sleep 0'"'"''
2020-04-19 02:29:07,878 p=14491 u=akokov |  <192.168.1.6> (0, b'ansible-tmp-1587252547.8363883-245698651101224=/root/.ansible/tmp/ansible-tmp-1587252547.8363883-245698651101224\n', b'OpenSSH_8.0p1 Ubuntu-6build1, OpenSSL 1.1.1c  28 May 2019\r\ndebug1: Reading configuration data /etc/ssh/ssh_config\r\ndebug1: /etc/ssh/ssh_config line 19: Applying options for *\r\ndebug2: resolve_canonicalize: hostname 192.168.1.6 is address\r\ndebug1: auto-mux: Trying existing master\r\ndebug2: fd 3 setting O_NONBLOCK\r\ndebug2: mux_client_hello_exchange: master version 4\r\ndebug3: mux_client_forwards: request forwardings: 0 local, 0 remote\r\ndebug3: mux_client_request_session: entering\r\ndebug3: mux_client_request_alive: entering\r\ndebug3: mux_client_request_alive: done pid = 14505\r\ndebug3: mux_client_request_session: session request sent\r\ndebug3: mux_client_read_packet: read header failed: Broken pipe\r\ndebug2: Received exit status from master 0\r\n')
2020-04-19 02:29:07,981 p=14491 u=akokov |  Using module file /usr/lib/python3/dist-packages/ansible/modules/net_tools/basics/get_url.py
2020-04-19 02:29:07,981 p=14491 u=akokov |  <192.168.1.6> PUT /home/akokov/.ansible/tmp/ansible-local-14491f0q6ghcl/tmps48qatw0 TO /root/.ansible/tmp/ansible-tmp-1587252547.8363883-245698651101224/AnsiballZ_get_url.py
2020-04-19 02:29:07,981 p=14491 u=akokov |  <192.168.1.6> SSH: EXEC sftp -b - -vvv -C -o ControlMaster=auto -o ControlPersist=60s -o KbdInteractiveAuthentication=no -o PreferredAuthentications=gssapi-with-mic,gssapi-keyex,hostbased,publickey -o PasswordAuthentication=no -o 'User="root"' -o ConnectTimeout=10 -o ControlPath=/home/akokov/.ansible/cp/c3e3deaa98 '[192.168.1.6]'
2020-04-19 02:29:08,011 p=14491 u=akokov |  <192.168.1.6> (0, b'sftp> put /home/akokov/.ansible/tmp/ansible-local-14491f0q6ghcl/tmps48qatw0 /root/.ansible/tmp/ansible-tmp-1587252547.8363883-245698651101224/AnsiballZ_get_url.py\n', b'OpenSSH_8.0p1 Ubuntu-6build1, OpenSSL 1.1.1c  28 May 2019\r\ndebug1: Reading configuration data /etc/ssh/ssh_config\r\ndebug1: /etc/ssh/ssh_config line 19: Applying options for *\r\ndebug2: resolve_canonicalize: hostname 192.168.1.6 is address\r\ndebug1: auto-mux: Trying existing master\r\ndebug2: fd 3 setting O_NONBLOCK\r\ndebug2: mux_client_hello_exchange: master version 4\r\ndebug3: mux_client_forwards: request forwardings: 0 local, 0 remote\r\ndebug3: mux_client_request_session: entering\r\ndebug3: mux_client_request_alive: entering\r\ndebug3: mux_client_request_alive: done pid = 14505\r\ndebug3: mux_client_request_session: session request sent\r\ndebug2: Remote version: 3\r\ndebug2: Server supports extension "posix-rename@openssh.com" revision 1\r\ndebug2: Server supports extension "statvfs@openssh.com" revision 2\r\ndebug2: Server supports extension "fstatvfs@openssh.com" revision 2\r\ndebug2: Server supports extension "hardlink@openssh.com" revision 1\r\ndebug2: Server supports extension "fsync@openssh.com" revision 1\r\ndebug2: Server supports extension "lsetstat@openssh.com" revision 1\r\ndebug3: Sent message fd 3 T:16 I:1\r\ndebug3: SSH_FXP_REALPATH . -> /root size 0\r\ndebug3: Looking up /home/akokov/.ansible/tmp/ansible-local-14491f0q6ghcl/tmps48qatw0\r\ndebug3: Sent message fd 3 T:17 I:2\r\ndebug3: Received stat reply T:101 I:2\r\ndebug1: Couldn\'t stat remote file: No such file or directory\r\ndebug3: Sent message SSH2_FXP_OPEN I:3 P:/root/.ansible/tmp/ansible-tmp-1587252547.8363883-245698651101224/AnsiballZ_get_url.py\r\ndebug3: Sent message SSH2_FXP_WRITE I:4 O:0 S:32768\r\ndebug3: SSH2_FXP_STATUS 0\r\ndebug3: In write loop, ack for 4 32768 bytes at 0\r\ndebug3: Sent message SSH2_FXP_WRITE I:5 O:32768 S:32768\r\ndebug3: Sent message SSH2_FXP_WRITE I:6 O:65536 S:32768\r\ndebug3: Sent message SSH2_FXP_WRITE I:7 O:98304 S:32768\r\ndebug3: Sent message SSH2_FXP_WRITE I:8 O:131072 S:896\r\ndebug3: SSH2_FXP_STATUS 0\r\ndebug3: In write loop, ack for 5 32768 bytes at 32768\r\ndebug3: SSH2_FXP_STATUS 0\r\ndebug3: In write loop, ack for 6 32768 bytes at 65536\r\ndebug3: SSH2_FXP_STATUS 0\r\ndebug3: In write loop, ack for 7 32768 bytes at 98304\r\ndebug3: SSH2_FXP_STATUS 0\r\ndebug3: In write loop, ack for 8 896 bytes at 131072\r\ndebug3: Sent message SSH2_FXP_CLOSE I:4\r\ndebug3: SSH2_FXP_STATUS 0\r\ndebug3: mux_client_read_packet: read header failed: Broken pipe\r\ndebug2: Received exit status from master 0\r\n')
2020-04-19 02:29:08,011 p=14491 u=akokov |  <192.168.1.6> ESTABLISH SSH CONNECTION FOR USER: root
2020-04-19 02:29:08,012 p=14491 u=akokov |  <192.168.1.6> SSH: EXEC ssh -vvv -C -o ControlMaster=auto -o ControlPersist=60s -o KbdInteractiveAuthentication=no -o PreferredAuthentications=gssapi-with-mic,gssapi-keyex,hostbased,publickey -o PasswordAuthentication=no -o 'User="root"' -o ConnectTimeout=10 -o ControlPath=/home/akokov/.ansible/cp/c3e3deaa98 192.168.1.6 '/bin/sh -c '"'"'chmod u+x /root/.ansible/tmp/ansible-tmp-1587252547.8363883-245698651101224/ /root/.ansible/tmp/ansible-tmp-1587252547.8363883-245698651101224/AnsiballZ_get_url.py && sleep 0'"'"''
2020-04-19 02:29:08,033 p=14491 u=akokov |  <192.168.1.6> (0, b'', b'OpenSSH_8.0p1 Ubuntu-6build1, OpenSSL 1.1.1c  28 May 2019\r\ndebug1: Reading configuration data /etc/ssh/ssh_config\r\ndebug1: /etc/ssh/ssh_config line 19: Applying options for *\r\ndebug2: resolve_canonicalize: hostname 192.168.1.6 is address\r\ndebug1: auto-mux: Trying existing master\r\ndebug2: fd 3 setting O_NONBLOCK\r\ndebug2: mux_client_hello_exchange: master version 4\r\ndebug3: mux_client_forwards: request forwardings: 0 local, 0 remote\r\ndebug3: mux_client_request_session: entering\r\ndebug3: mux_client_request_alive: entering\r\ndebug3: mux_client_request_alive: done pid = 14505\r\ndebug3: mux_client_request_session: session request sent\r\ndebug3: mux_client_read_packet: read header failed: Broken pipe\r\ndebug2: Received exit status from master 0\r\n')
2020-04-19 02:29:08,033 p=14491 u=akokov |  <192.168.1.6> ESTABLISH SSH CONNECTION FOR USER: root
2020-04-19 02:29:08,034 p=14491 u=akokov |  <192.168.1.6> SSH: EXEC ssh -vvv -C -o ControlMaster=auto -o ControlPersist=60s -o KbdInteractiveAuthentication=no -o PreferredAuthentications=gssapi-with-mic,gssapi-keyex,hostbased,publickey -o PasswordAuthentication=no -o 'User="root"' -o ConnectTimeout=10 -o ControlPath=/home/akokov/.ansible/cp/c3e3deaa98 -tt 192.168.1.6 '/bin/sh -c '"'"'/usr/libexec/platform-python /root/.ansible/tmp/ansible-tmp-1587252547.8363883-245698651101224/AnsiballZ_get_url.py && sleep 0'"'"''
2020-04-19 02:29:12,565 p=14491 u=akokov |  <192.168.1.6> (1, b'\r\n{"msg": "Request failed", "status_code": 404, "response": "HTTP Error 404: Not Found", "url": "http://download.oracle.com/otn-pub/java/jdk/8u60-b27/jre-8u60-linux-x64.rpm", "dest": "/opt/jre-8u60-linux-x64.rpm", "elapsed": 4, "failed": true, "invocation": {"module_args": {"url": "http://download.oracle.com/otn-pub/java/jdk/8u60-b27/jre-8u60-linux-x64.rpm", "dest": "/opt/jre-8u60-linux-x64.rpm", "headers": "Cookie: gpw_e24=http%3A%2F%2Fwww.oracle.com%2F; oraclelicense=accept-securebackup-cookie", "force": false, "http_agent": "ansible-httpget", "use_proxy": true, "validate_certs": true, "force_basic_auth": false, "sha256sum": "", "checksum": "", "timeout": 10, "follow": false, "url_username": null, "url_password": null, "client_cert": null, "client_key": null, "backup": null, "tmp_dest": null, "mode": null, "owner": null, "group": null, "seuser": null, "serole": null, "selevel": null, "setype": null, "attributes": null, "src": null, "content": null, "remote_src": null, "regexp": null, "delimiter": null, "directory_mode": null, "unsafe_writes": null}}, "deprecations": [{"msg": "Supplying `headers` as a string is deprecated. Please use dict/hash format for `headers`", "version": "2.10"}]}\r\n', b'OpenSSH_8.0p1 Ubuntu-6build1, OpenSSL 1.1.1c  28 May 2019\r\ndebug1: Reading configuration data /etc/ssh/ssh_config\r\ndebug1: /etc/ssh/ssh_config line 19: Applying options for *\r\ndebug2: resolve_canonicalize: hostname 192.168.1.6 is address\r\ndebug1: auto-mux: Trying existing master\r\ndebug2: fd 3 setting O_NONBLOCK\r\ndebug2: mux_client_hello_exchange: master version 4\r\ndebug3: mux_client_forwards: request forwardings: 0 local, 0 remote\r\ndebug3: mux_client_request_session: entering\r\ndebug3: mux_client_request_alive: entering\r\ndebug3: mux_client_request_alive: done pid = 14505\r\ndebug3: mux_client_request_session: session request sent\r\ndebug3: mux_client_read_packet: read header failed: Broken pipe\r\ndebug2: Received exit status from master 1\r\nShared connection to 192.168.1.6 closed.\r\n')
2020-04-19 02:29:12,565 p=14491 u=akokov |  <192.168.1.6> Failed to connect to the host via ssh: OpenSSH_8.0p1 Ubuntu-6build1, OpenSSL 1.1.1c  28 May 2019
debug1: Reading configuration data /etc/ssh/ssh_config
debug1: /etc/ssh/ssh_config line 19: Applying options for *
debug2: resolve_canonicalize: hostname 192.168.1.6 is address
debug1: auto-mux: Trying existing master
debug2: fd 3 setting O_NONBLOCK
debug2: mux_client_hello_exchange: master version 4
debug3: mux_client_forwards: request forwardings: 0 local, 0 remote
debug3: mux_client_request_session: entering
debug3: mux_client_request_alive: entering
debug3: mux_client_request_alive: done pid = 14505
debug3: mux_client_request_session: session request sent
debug3: mux_client_read_packet: read header failed: Broken pipe
debug2: Received exit status from master 1
Shared connection to 192.168.1.6 closed.
2020-04-19 02:29:12,566 p=14491 u=akokov |  <192.168.1.6> ESTABLISH SSH CONNECTION FOR USER: root
2020-04-19 02:29:12,566 p=14491 u=akokov |  <192.168.1.6> SSH: EXEC ssh -vvv -C -o ControlMaster=auto -o ControlPersist=60s -o KbdInteractiveAuthentication=no -o PreferredAuthentications=gssapi-with-mic,gssapi-keyex,hostbased,publickey -o PasswordAuthentication=no -o 'User="root"' -o ConnectTimeout=10 -o ControlPath=/home/akokov/.ansible/cp/c3e3deaa98 192.168.1.6 '/bin/sh -c '"'"'rm -f -r /root/.ansible/tmp/ansible-tmp-1587252547.8363883-245698651101224/ > /dev/null 2>&1 && sleep 0'"'"''
2020-04-19 02:29:12,588 p=14491 u=akokov |  <192.168.1.6> (0, b'', b'OpenSSH_8.0p1 Ubuntu-6build1, OpenSSL 1.1.1c  28 May 2019\r\ndebug1: Reading configuration data /etc/ssh/ssh_config\r\ndebug1: /etc/ssh/ssh_config line 19: Applying options for *\r\ndebug2: resolve_canonicalize: hostname 192.168.1.6 is address\r\ndebug1: auto-mux: Trying existing master\r\ndebug2: fd 3 setting O_NONBLOCK\r\ndebug2: mux_client_hello_exchange: master version 4\r\ndebug3: mux_client_forwards: request forwardings: 0 local, 0 remote\r\ndebug3: mux_client_request_session: entering\r\ndebug3: mux_client_request_alive: entering\r\ndebug3: mux_client_request_alive: done pid = 14505\r\ndebug3: mux_client_request_session: session request sent\r\ndebug3: mux_client_read_packet: read header failed: Broken pipe\r\ndebug2: Received exit status from master 0\r\n')
2020-04-19 02:29:12,590 p=14491 u=akokov |  [DEPRECATION WARNING]: Supplying `headers` as a string is deprecated. Please use dict/hash format for `headers`. This feature will be removed in version 2.10. Deprecation warnings can be disabled by setting 
deprecation_warnings=False in ansible.cfg.
2020-04-19 02:29:12,591 p=14491 u=akokov |  fatal: [test]: FAILED! => {
    "changed": false,
    "dest": "/opt/jre-8u60-linux-x64.rpm",
    "elapsed": 4,
    "invocation": {
        "module_args": {
            "attributes": null,
            "backup": null,
            "checksum": "",
            "client_cert": null,
            "client_key": null,
            "content": null,
            "delimiter": null,
            "dest": "/opt/jre-8u60-linux-x64.rpm",
            "directory_mode": null,
            "follow": false,
            "force": false,
            "force_basic_auth": false,
            "group": null,
            "headers": "Cookie: gpw_e24=http%3A%2F%2Fwww.oracle.com%2F; oraclelicense=accept-securebackup-cookie",
            "http_agent": "ansible-httpget",
            "mode": null,
            "owner": null,
            "regexp": null,
            "remote_src": null,
            "selevel": null,
            "serole": null,
            "setype": null,
            "seuser": null,
            "sha256sum": "",
            "src": null,
            "timeout": 10,
            "tmp_dest": null,
            "unsafe_writes": null,
            "url": "http://download.oracle.com/otn-pub/java/jdk/8u60-b27/jre-8u60-linux-x64.rpm",
            "url_password": null,
            "url_username": null,
            "use_proxy": true,
            "validate_certs": true
        }
    },
    "msg": "Request failed",
    "response": "HTTP Error 404: Not Found",
    "status_code": 404,
    "url": "http://download.oracle.com/otn-pub/java/jdk/8u60-b27/jre-8u60-linux-x64.rpm"
}
2020-04-19 02:29:12,591 p=14491 u=akokov |  PLAY RECAP **********************************************************************************************************************************************************************************************************
2020-04-19 02:29:12,592 p=14491 u=akokov |  test                       : ok=1    changed=0    unreachable=0    failed=1    skipped=0    rescued=0    ignored=0   
2020-04-19 02:29:28,108 p=14558 u=akokov |  ansible-playbook 2.8.3
  config file = /home/ansible/ansible.cfg
  configured module search path = ['/home/akokov/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/lib/python3/dist-packages/ansible
  executable location = /usr/bin/ansible-playbook
  python version = 3.7.5 (default, Nov 20 2019, 09:21:52) [GCC 9.2.1 20191008]
2020-04-19 02:29:28,108 p=14558 u=akokov |  Using /home/ansible/ansible.cfg as config file
2020-04-19 02:29:28,126 p=14558 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/docker.py) as it seems to be invalid: No module named 'distutils.spawn'

2020-04-19 02:29:28,159 p=14558 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/iocage.py) as it seems to be invalid: No module named 'distutils.spawn'

2020-04-19 02:29:28,159 p=14558 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/jail.py) as it seems to be invalid: No module named 'distutils.spawn'

2020-04-19 02:29:28,159 p=14558 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/kubectl.py) as it seems to be invalid: No module named 'distutils.spawn'

2020-04-19 02:29:28,159 p=14558 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/libvirt_lxc.py) as it seems to be invalid: No module named 'distutils.spawn'

2020-04-19 02:29:28,161 p=14558 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/lxd.py) as it seems to be invalid: No module named 'distutils.spawn'

2020-04-19 02:29:28,167 p=14558 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/oc.py) as it seems to be invalid: No module named 'distutils.spawn'

2020-04-19 02:29:28,189 p=14558 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/zone.py) as it seems to be invalid: No module named 'distutils.spawn'

2020-04-19 02:29:28,299 p=14558 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/callback/osx_say.py) as it seems to be invalid: No module named 'distutils.spawn'

2020-04-19 02:29:28,300 p=14558 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/callback/say.py) as it seems to be invalid: No module named 'distutils.spawn'

2020-04-19 02:29:28,307 p=14558 u=akokov |  PLAYBOOK: playbook-install.yml **************************************************************************************************************************************************************************************
2020-04-19 02:29:28,307 p=14558 u=akokov |  1 plays in playbook-install.yml
2020-04-19 02:29:28,327 p=14558 u=akokov |  PLAY [test] *********************************************************************************************************************************************************************************************************
2020-04-19 02:29:28,333 p=14558 u=akokov |  TASK [Gathering Facts] **********************************************************************************************************************************************************************************************
2020-04-19 02:29:28,333 p=14558 u=akokov |  task path: /home/ansible/playbook-install.yml:3
2020-04-19 02:29:29,187 p=14558 u=akokov |  ok: [test]
2020-04-19 02:29:29,187 p=14558 u=akokov |  META: ran handlers
2020-04-19 02:29:29,193 p=14558 u=akokov |  TASK [Download Java JRE] ********************************************************************************************************************************************************************************************
2020-04-19 02:29:29,193 p=14558 u=akokov |  task path: /home/ansible/playbook-install.yml:11
2020-04-19 02:29:30,831 p=14558 u=akokov |  [DEPRECATION WARNING]: Supplying `headers` as a string is deprecated. Please use dict/hash format for `headers`. This feature will be removed in version 2.10. Deprecation warnings can be disabled by setting 
deprecation_warnings=False in ansible.cfg.
2020-04-19 02:29:30,832 p=14558 u=akokov |  fatal: [test]: FAILED! => {"changed": false, "dest": "/opt/jre-8u60-linux-x64.rpm", "elapsed": 1, "msg": "Request failed", "response": "HTTP Error 404: Not Found", "status_code": 404, "url": "http://download.oracle.com/otn-pub/java/jdk/8u60-b27/jre-8u60-linux-x64.rpm"}
2020-04-19 02:29:30,833 p=14558 u=akokov |  PLAY RECAP **********************************************************************************************************************************************************************************************************
2020-04-19 02:29:30,833 p=14558 u=akokov |  test                       : ok=1    changed=0    unreachable=0    failed=1    skipped=0    rescued=0    ignored=0   
2020-04-19 15:07:50,337 p=26143 u=akokov |  ansible-playbook 2.8.3
  config file = /home/ansible/ansible.cfg
  configured module search path = ['/home/akokov/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/lib/python3/dist-packages/ansible
  executable location = /usr/bin/ansible-playbook
  python version = 3.7.5 (default, Nov 20 2019, 09:21:52) [GCC 9.2.1 20191008]
2020-04-19 15:07:50,338 p=26143 u=akokov |  Using /home/ansible/ansible.cfg as config file
2020-04-19 15:07:50,363 p=26143 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/docker.py) as it seems to be invalid: No module named 'distutils.spawn'

2020-04-19 15:07:50,411 p=26143 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/iocage.py) as it seems to be invalid: No module named 'distutils.spawn'

2020-04-19 15:07:50,411 p=26143 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/jail.py) as it seems to be invalid: No module named 'distutils.spawn'

2020-04-19 15:07:50,412 p=26143 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/kubectl.py) as it seems to be invalid: No module named 'distutils.spawn'

2020-04-19 15:07:50,412 p=26143 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/libvirt_lxc.py) as it seems to be invalid: No module named 'distutils.spawn'

2020-04-19 15:07:50,414 p=26143 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/lxd.py) as it seems to be invalid: No module named 'distutils.spawn'

2020-04-19 15:07:50,422 p=26143 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/oc.py) as it seems to be invalid: No module named 'distutils.spawn'

2020-04-19 15:07:50,482 p=26143 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/zone.py) as it seems to be invalid: No module named 'distutils.spawn'

2020-04-19 15:07:50,653 p=26143 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/callback/osx_say.py) as it seems to be invalid: No module named 'distutils.spawn'

2020-04-19 15:07:50,654 p=26143 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/callback/say.py) as it seems to be invalid: No module named 'distutils.spawn'

2020-04-19 15:07:50,663 p=26143 u=akokov |  PLAYBOOK: playbook-install.yml **************************************************************************************************************************************************************************************
2020-04-19 15:07:50,663 p=26143 u=akokov |  1 plays in playbook-install.yml
2020-04-19 15:07:50,692 p=26143 u=akokov |  PLAY [test] *********************************************************************************************************************************************************************************************************
2020-04-19 15:07:50,698 p=26143 u=akokov |  TASK [Gathering Facts] **********************************************************************************************************************************************************************************************
2020-04-19 15:07:50,698 p=26143 u=akokov |  task path: /home/ansible/playbook-install.yml:3
2020-04-19 15:07:51,900 p=26143 u=akokov |  ok: [test]
2020-04-19 15:07:51,900 p=26143 u=akokov |  META: ran handlers
2020-04-19 15:07:51,907 p=26143 u=akokov |  TASK [install Java-JDK v1.8] ****************************************************************************************************************************************************************************************
2020-04-19 15:07:51,907 p=26143 u=akokov |  task path: /home/ansible/playbook-install.yml:11
2020-04-19 15:08:06,019 p=26143 u=akokov |  changed: [test] => {"changed": true, "msg": "", "rc": 0, "results": ["Installed: java-1.8.0-openjdk-devel", "Installed: libXfixes-5.0.3-7.el8.x86_64", "Installed: java-1.8.0-openjdk-devel-1:1.8.0.242.b08-0.el8_1.x86_64", "Installed: harfbuzz-1.7.5-3.el8.x86_64", "Installed: java-1.8.0-openjdk-headless-1:1.8.0.242.b08-0.el8_1.x86_64", "Installed: libXft-2.3.2-10.el8.x86_64", "Installed: libXi-1.7.9-7.el8.x86_64", "Installed: libXinerama-1.1.4-1.el8.x86_64", "Installed: libdatrie-0.2.9-7.el8.x86_64", "Installed: hicolor-icon-theme-0.17-2.el8.noarch", "Installed: lksctp-tools-1.0.18-3.el8.x86_64", "Installed: tzdata-java-2019c-1.el8.noarch", "Installed: graphite2-1.3.10-10.el8.x86_64", "Installed: jasper-libs-2.0.14-4.el8.x86_64", "Installed: libXrandr-1.5.1-7.el8.x86_64", "Installed: alsa-lib-1.1.9-4.el8.x86_64", "Installed: gdk-pixbuf2-modules-2.36.12-5.el8.x86_64", "Installed: javapackages-filesystem-5.3.0-1.module_el8.0.0+11+5b8c10bd.noarch", "Installed: libXtst-1.2.3-7.el8.x86_64", "Installed: jbigkit-libs-2.1-14.el8.x86_64", "Installed: libthai-0.1.27-2.el8.x86_64", "Installed: xorg-x11-font-utils-1:7.5-40.el8.x86_64", "Installed: libtiff-4.0.9-15.el8.x86_64", "Installed: gtk-update-icon-cache-3.22.30-4.el8.x86_64", "Installed: libjpeg-turbo-1.5.3-10.el8.x86_64", "Installed: xorg-x11-fonts-Type1-7.5-19.el8.noarch", "Installed: gtk2-2.24.32-4.el8.x86_64", "Installed: fribidi-1.0.4-7.el8_1.x86_64", "Installed: libfontenc-1.1.3-8.el8.x86_64", "Installed: giflib-5.1.4-3.el8.x86_64", "Installed: pango-1.42.4-6.el8.x86_64", "Installed: copy-jdk-configs-3.7-1.el8.noarch", "Installed: lua-5.3.4-11.el8.x86_64", "Installed: ttmkfdir-3.0.9-54.el8.x86_64", "Installed: libXcomposite-0.4.4-14.el8.x86_64", "Installed: libXcursor-1.1.15-3.el8.x86_64", "Installed: libXdamage-1.1.4-14.el8.x86_64", "Installed: java-1.8.0-openjdk-1:1.8.0.242.b08-0.el8_1.x86_64", "Installed: atk-2.28.1-1.el8.x86_64"]}
2020-04-19 15:08:06,026 p=26143 u=akokov |  TASK [install the latest version of Apache Maven] *******************************************************************************************************************************************************************
2020-04-19 15:08:06,026 p=26143 u=akokov |  task path: /home/ansible/playbook-install.yml:18
2020-04-19 15:08:11,978 p=26143 u=akokov |  changed: [test] => {"changed": true, "msg": "", "rc": 0, "results": ["Installed: maven", "Installed: guava20-20.0-8.module_el8.0.0+39+6a9b6e22.noarch", "Installed: sisu-inject-1:0.3.3-6.module_el8.0.0+39+6a9b6e22.noarch", "Installed: sisu-plexus-1:0.3.3-6.module_el8.0.0+39+6a9b6e22.noarch", "Installed: cdi-api-1.2-8.module_el8.0.0+39+6a9b6e22.noarch", "Installed: jansi-1.17.1-1.module_el8.0.0+82+8ee6c375.noarch", "Installed: slf4j-1.7.25-4.module_el8.0.0+39+6a9b6e22.noarch", "Installed: jansi-native-1.7-7.module_el8.0.0+39+6a9b6e22.x86_64", "Installed: plexus-cipher-1.7-14.module_el8.0.0+39+6a9b6e22.noarch", "Installed: plexus-classworlds-2.5.2-9.module_el8.0.0+39+6a9b6e22.noarch", "Installed: glassfish-el-api-3.0.1-0.7.b08.module_el8.0.0+39+6a9b6e22.noarch", "Installed: plexus-containers-component-annotations-1.7.1-8.module_el8.0.0+39+6a9b6e22.noarch", "Installed: plexus-interpolation-1.22-9.module_el8.0.0+39+6a9b6e22.noarch", "Installed: plexus-sec-dispatcher-1.4-26.module_el8.0.0+39+6a9b6e22.noarch", "Installed: plexus-utils-3.1.0-3.module_el8.0.0+39+6a9b6e22.noarch", "Installed: atinject-1-28.20100611svn86.module_el8.0.0+39+6a9b6e22.noarch", "Installed: google-guice-4.1-11.module_el8.0.0+39+6a9b6e22.noarch", "Installed: hawtjni-runtime-1.16-2.module_el8.0.0+39+6a9b6e22.noarch", "Installed: maven-1:3.5.4-5.module_el8.0.0+39+6a9b6e22.noarch", "Installed: maven-lib-1:3.5.4-5.module_el8.0.0+39+6a9b6e22.noarch", "Installed: maven-resolver-api-1:1.1.1-2.module_el8.0.0+39+6a9b6e22.noarch", "Installed: maven-resolver-connector-basic-1:1.1.1-2.module_el8.0.0+39+6a9b6e22.noarch", "Installed: maven-resolver-impl-1:1.1.1-2.module_el8.0.0+39+6a9b6e22.noarch", "Installed: maven-resolver-spi-1:1.1.1-2.module_el8.0.0+39+6a9b6e22.noarch", "Installed: maven-resolver-transport-wagon-1:1.1.1-2.module_el8.0.0+39+6a9b6e22.noarch", "Installed: maven-resolver-util-1:1.1.1-2.module_el8.0.0+39+6a9b6e22.noarch", "Installed: maven-shared-utils-3.2.1-0.1.module_el8.0.0+39+6a9b6e22.noarch", "Installed: maven-wagon-file-3.1.0-1.module_el8.0.0+39+6a9b6e22.noarch", "Installed: maven-wagon-http-3.1.0-1.module_el8.0.0+39+6a9b6e22.noarch", "Installed: maven-wagon-http-shared-3.1.0-1.module_el8.0.0+39+6a9b6e22.noarch", "Installed: maven-wagon-provider-api-3.1.0-1.module_el8.0.0+39+6a9b6e22.noarch", "Installed: httpcomponents-client-4.5.5-4.module_el8.0.0+39+6a9b6e22.noarch", "Installed: httpcomponents-core-4.4.10-3.module_el8.0.0+39+6a9b6e22.noarch", "Installed: javapackages-tools-5.3.0-1.module_el8.0.0+11+5b8c10bd.noarch", "Installed: jboss-interceptors-1.2-api-1.0.0-8.module_el8.0.0+39+6a9b6e22.noarch", "Installed: jcl-over-slf4j-1.7.25-4.module_el8.0.0+39+6a9b6e22.noarch", "Installed: jsoup-1.11.3-3.module_el8.0.0+39+6a9b6e22.noarch", "Installed: geronimo-annotation-1.0-23.module_el8.0.0+39+6a9b6e22.noarch", "Installed: aopalliance-1.0-17.module_el8.0.0+39+6a9b6e22.noarch", "Installed: apache-commons-cli-1.4-4.module_el8.0.0+39+6a9b6e22.noarch", "Installed: apache-commons-codec-1.11-3.module_el8.0.0+39+6a9b6e22.noarch", "Installed: apache-commons-io-1:2.6-3.module_el8.0.0+39+6a9b6e22.noarch", "Installed: apache-commons-lang3-3.7-3.module_el8.0.0+39+6a9b6e22.noarch", "Installed: apache-commons-logging-1.2-13.module_el8.0.0+39+6a9b6e22.noarch", "Installed: publicsuffix-list-20180723-1.el8.noarch"]}
2020-04-19 15:08:11,986 p=26143 u=akokov |  TASK [install the latest version of Docker] *************************************************************************************************************************************************************************
2020-04-19 15:08:11,986 p=26143 u=akokov |  task path: /home/ansible/playbook-install.yml:25
2020-04-19 15:08:12,922 p=26143 u=akokov |  fatal: [test]: FAILED! => {"changed": false, "failures": ["No package docker available."], "msg": "Failed to install some of the specified packages", "rc": 1, "results": []}
2020-04-19 15:08:12,923 p=26143 u=akokov |  PLAY RECAP **********************************************************************************************************************************************************************************************************
2020-04-19 15:08:12,923 p=26143 u=akokov |  test                       : ok=3    changed=2    unreachable=0    failed=1    skipped=0    rescued=0    ignored=0   
2020-04-19 15:29:57,751 p=29996 u=akokov |  ansible-playbook 2.8.3
  config file = /home/ansible/ansible.cfg
  configured module search path = ['/home/akokov/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/lib/python3/dist-packages/ansible
  executable location = /usr/bin/ansible-playbook
  python version = 3.7.5 (default, Nov 20 2019, 09:21:52) [GCC 9.2.1 20191008]
2020-04-19 15:29:57,751 p=29996 u=akokov |  Using /home/ansible/ansible.cfg as config file
2020-04-19 15:29:57,770 p=29996 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/docker.py) as it seems to be invalid: No module named 'distutils.spawn'

2020-04-19 15:29:57,804 p=29996 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/iocage.py) as it seems to be invalid: No module named 'distutils.spawn'

2020-04-19 15:29:57,804 p=29996 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/jail.py) as it seems to be invalid: No module named 'distutils.spawn'

2020-04-19 15:29:57,804 p=29996 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/kubectl.py) as it seems to be invalid: No module named 'distutils.spawn'

2020-04-19 15:29:57,805 p=29996 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/libvirt_lxc.py) as it seems to be invalid: No module named 'distutils.spawn'

2020-04-19 15:29:57,806 p=29996 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/lxd.py) as it seems to be invalid: No module named 'distutils.spawn'

2020-04-19 15:29:57,812 p=29996 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/oc.py) as it seems to be invalid: No module named 'distutils.spawn'

2020-04-19 15:29:57,834 p=29996 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/zone.py) as it seems to be invalid: No module named 'distutils.spawn'

2020-04-19 15:29:57,946 p=29996 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/callback/osx_say.py) as it seems to be invalid: No module named 'distutils.spawn'

2020-04-19 15:29:57,947 p=29996 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/callback/say.py) as it seems to be invalid: No module named 'distutils.spawn'

2020-04-19 15:29:57,954 p=29996 u=akokov |  PLAYBOOK: playbook-install.yml **************************************************************************************************************************************************************************************
2020-04-19 15:29:57,954 p=29996 u=akokov |  1 plays in playbook-install.yml
2020-04-19 15:29:57,975 p=29996 u=akokov |  PLAY [test] *********************************************************************************************************************************************************************************************************
2020-04-19 15:29:57,980 p=29996 u=akokov |  TASK [Gathering Facts] **********************************************************************************************************************************************************************************************
2020-04-19 15:29:57,980 p=29996 u=akokov |  task path: /home/ansible/playbook-install.yml:3
2020-04-19 15:29:59,044 p=29996 u=akokov |  ok: [test]
2020-04-19 15:29:59,045 p=29996 u=akokov |  META: ran handlers
2020-04-19 15:29:59,051 p=29996 u=akokov |  TASK [install latest Yum-Utils] *************************************************************************************************************************************************************************************
2020-04-19 15:29:59,051 p=29996 u=akokov |  task path: /home/ansible/playbook-install.yml:11
2020-04-19 15:30:00,892 p=29996 u=akokov |  changed: [test] => {"changed": true, "msg": "", "rc": 0, "results": ["Installed: yum-utils", "Installed: yum-utils-4.0.8-3.el8.noarch"]}
2020-04-19 15:30:00,898 p=29996 u=akokov |  TASK [install Java-JDK v1.8] ****************************************************************************************************************************************************************************************
2020-04-19 15:30:00,899 p=29996 u=akokov |  task path: /home/ansible/playbook-install.yml:18
2020-04-19 15:30:01,795 p=29996 u=akokov |  ok: [test] => {"changed": false, "msg": "Nothing to do", "rc": 0, "results": ["Installed: java-1.8.0-openjdk-devel"]}
2020-04-19 15:30:01,802 p=29996 u=akokov |  TASK [install the latest version of Apache Maven] *******************************************************************************************************************************************************************
2020-04-19 15:30:01,803 p=29996 u=akokov |  task path: /home/ansible/playbook-install.yml:25
2020-04-19 15:30:02,690 p=29996 u=akokov |  ok: [test] => {"changed": false, "msg": "Nothing to do", "rc": 0, "results": ["Installed: maven"]}
2020-04-19 15:30:02,697 p=29996 u=akokov |  TASK [install the latest version of Docker] *************************************************************************************************************************************************************************
2020-04-19 15:30:02,697 p=29996 u=akokov |  task path: /home/ansible/playbook-install.yml:32
2020-04-19 15:30:03,611 p=29996 u=akokov |  fatal: [test]: FAILED! => {"changed": false, "failures": ["No package docker available."], "msg": "Failed to install some of the specified packages", "rc": 1, "results": []}
2020-04-19 15:30:03,612 p=29996 u=akokov |  PLAY RECAP **********************************************************************************************************************************************************************************************************
2020-04-19 15:30:03,612 p=29996 u=akokov |  test                       : ok=4    changed=1    unreachable=0    failed=1    skipped=0    rescued=0    ignored=0   
2020-04-19 15:46:44,629 p=32599 u=akokov |  ansible-playbook 2.8.3
  config file = /home/ansible/ansible.cfg
  configured module search path = ['/home/akokov/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/lib/python3/dist-packages/ansible
  executable location = /usr/bin/ansible-playbook
  python version = 3.7.5 (default, Nov 20 2019, 09:21:52) [GCC 9.2.1 20191008]
2020-04-19 15:46:44,629 p=32599 u=akokov |  Using /home/ansible/ansible.cfg as config file
2020-04-19 15:46:44,648 p=32599 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/docker.py) as it
seems to be invalid: No module named 'distutils.spawn'

2020-04-19 15:46:44,682 p=32599 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/iocage.py) as it
seems to be invalid: No module named 'distutils.spawn'

2020-04-19 15:46:44,682 p=32599 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/jail.py) as it seems
to be invalid: No module named 'distutils.spawn'

2020-04-19 15:46:44,682 p=32599 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/kubectl.py) as it
seems to be invalid: No module named 'distutils.spawn'

2020-04-19 15:46:44,682 p=32599 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/libvirt_lxc.py) as it
seems to be invalid: No module named 'distutils.spawn'

2020-04-19 15:46:44,684 p=32599 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/lxd.py) as it seems
to be invalid: No module named 'distutils.spawn'

2020-04-19 15:46:44,690 p=32599 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/oc.py) as it seems to
be invalid: No module named 'distutils.spawn'

2020-04-19 15:46:44,713 p=32599 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/zone.py) as it seems
to be invalid: No module named 'distutils.spawn'

2020-04-19 15:46:44,830 p=32599 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/callback/osx_say.py) as it seems
to be invalid: No module named 'distutils.spawn'

2020-04-19 15:46:44,831 p=32599 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/callback/say.py) as it seems to
be invalid: No module named 'distutils.spawn'

2020-04-19 15:46:44,838 p=32599 u=akokov |  PLAYBOOK: playbook-install.yml ******************************************************************************
2020-04-19 15:46:44,838 p=32599 u=akokov |  1 plays in playbook-install.yml
2020-04-19 15:46:44,859 p=32599 u=akokov |  PLAY [test] *************************************************************************************************
2020-04-19 15:46:44,865 p=32599 u=akokov |  TASK [Gathering Facts] **************************************************************************************
2020-04-19 15:46:44,865 p=32599 u=akokov |  task path: /home/ansible/playbook-install.yml:3
2020-04-19 15:46:45,930 p=32599 u=akokov |  ok: [test]
2020-04-19 15:46:45,931 p=32599 u=akokov |  META: ran handlers
2020-04-19 15:46:45,937 p=32599 u=akokov |  TASK [install latest Yum-Utils] *****************************************************************************
2020-04-19 15:46:45,937 p=32599 u=akokov |  task path: /home/ansible/playbook-install.yml:11
2020-04-19 15:46:46,879 p=32599 u=akokov |  ok: [test] => {"changed": false, "msg": "Nothing to do", "rc": 0, "results": ["Installed: yum-utils"]}
2020-04-19 15:46:46,886 p=32599 u=akokov |  TASK [install Java-JDK v1.8] ********************************************************************************
2020-04-19 15:46:46,886 p=32599 u=akokov |  task path: /home/ansible/playbook-install.yml:18
2020-04-19 15:46:47,719 p=32599 u=akokov |  ok: [test] => {"changed": false, "msg": "Nothing to do", "rc": 0, "results": ["Installed: java-1.8.0-openjdk-devel"]}
2020-04-19 15:46:47,726 p=32599 u=akokov |  TASK [install the latest version of Apache Maven] ***********************************************************
2020-04-19 15:46:47,726 p=32599 u=akokov |  task path: /home/ansible/playbook-install.yml:25
2020-04-19 15:46:48,571 p=32599 u=akokov |  ok: [test] => {"changed": false, "msg": "Nothing to do", "rc": 0, "results": ["Installed: maven"]}
2020-04-19 15:46:48,578 p=32599 u=akokov |  TASK [add repository docker] ********************************************************************************
2020-04-19 15:46:48,578 p=32599 u=akokov |  task path: /home/ansible/playbook-install.yml:32
2020-04-19 15:46:48,930 p=32599 u=akokov |  fatal: [test]: FAILED! => {"changed": false, "msg": "Parameter 'description' is required."}
2020-04-19 15:46:48,931 p=32599 u=akokov |  PLAY RECAP **************************************************************************************************
2020-04-19 15:46:48,931 p=32599 u=akokov |  test                       : ok=4    changed=0    unreachable=0    failed=1    skipped=0    rescued=0    ignored=0   
2020-04-19 15:47:37,502 p=334 u=akokov |  ansible-playbook 2.8.3
  config file = /home/ansible/ansible.cfg
  configured module search path = ['/home/akokov/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/lib/python3/dist-packages/ansible
  executable location = /usr/bin/ansible-playbook
  python version = 3.7.5 (default, Nov 20 2019, 09:21:52) [GCC 9.2.1 20191008]
2020-04-19 15:47:37,502 p=334 u=akokov |  Using /home/ansible/ansible.cfg as config file
2020-04-19 15:47:37,521 p=334 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/docker.py) as it
seems to be invalid: No module named 'distutils.spawn'

2020-04-19 15:47:37,553 p=334 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/iocage.py) as it
seems to be invalid: No module named 'distutils.spawn'

2020-04-19 15:47:37,553 p=334 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/jail.py) as it seems
to be invalid: No module named 'distutils.spawn'

2020-04-19 15:47:37,554 p=334 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/kubectl.py) as it
seems to be invalid: No module named 'distutils.spawn'

2020-04-19 15:47:37,554 p=334 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/libvirt_lxc.py) as it
seems to be invalid: No module named 'distutils.spawn'

2020-04-19 15:47:37,555 p=334 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/lxd.py) as it seems
to be invalid: No module named 'distutils.spawn'

2020-04-19 15:47:37,561 p=334 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/oc.py) as it seems to
be invalid: No module named 'distutils.spawn'

2020-04-19 15:47:37,583 p=334 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/zone.py) as it seems
to be invalid: No module named 'distutils.spawn'

2020-04-19 15:47:37,696 p=334 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/callback/osx_say.py) as it seems
to be invalid: No module named 'distutils.spawn'

2020-04-19 15:47:37,697 p=334 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/callback/say.py) as it seems to
be invalid: No module named 'distutils.spawn'

2020-04-19 15:47:37,704 p=334 u=akokov |  PLAYBOOK: playbook-install.yml ******************************************************************************
2020-04-19 15:47:37,704 p=334 u=akokov |  1 plays in playbook-install.yml
2020-04-19 15:47:37,725 p=334 u=akokov |  PLAY [test] *************************************************************************************************
2020-04-19 15:47:37,730 p=334 u=akokov |  TASK [Gathering Facts] **************************************************************************************
2020-04-19 15:47:37,730 p=334 u=akokov |  task path: /home/ansible/playbook-install.yml:3
2020-04-19 15:47:38,592 p=334 u=akokov |  ok: [test]
2020-04-19 15:47:38,593 p=334 u=akokov |  META: ran handlers
2020-04-19 15:47:38,600 p=334 u=akokov |  TASK [install latest Yum-Utils] *****************************************************************************
2020-04-19 15:47:38,600 p=334 u=akokov |  task path: /home/ansible/playbook-install.yml:11
2020-04-19 15:47:39,560 p=334 u=akokov |  ok: [test] => {"changed": false, "msg": "Nothing to do", "rc": 0, "results": ["Installed: yum-utils"]}
2020-04-19 15:47:39,567 p=334 u=akokov |  TASK [install Java-JDK v1.8] ********************************************************************************
2020-04-19 15:47:39,567 p=334 u=akokov |  task path: /home/ansible/playbook-install.yml:18
2020-04-19 15:47:40,399 p=334 u=akokov |  ok: [test] => {"changed": false, "msg": "Nothing to do", "rc": 0, "results": ["Installed: java-1.8.0-openjdk-devel"]}
2020-04-19 15:47:40,406 p=334 u=akokov |  TASK [install the latest version of Apache Maven] ***********************************************************
2020-04-19 15:47:40,406 p=334 u=akokov |  task path: /home/ansible/playbook-install.yml:25
2020-04-19 15:47:41,252 p=334 u=akokov |  ok: [test] => {"changed": false, "msg": "Nothing to do", "rc": 0, "results": ["Installed: maven"]}
2020-04-19 15:47:41,258 p=334 u=akokov |  TASK [add repository docker] ********************************************************************************
2020-04-19 15:47:41,258 p=334 u=akokov |  task path: /home/ansible/playbook-install.yml:32
2020-04-19 15:47:41,605 p=334 u=akokov |  changed: [test] => {"changed": true, "repo": "docker-ce", "state": "present"}
2020-04-19 15:47:41,611 p=334 u=akokov |  TASK [install the latest version of Docker] *****************************************************************
2020-04-19 15:47:41,611 p=334 u=akokov |  task path: /home/ansible/playbook-install.yml:38
2020-04-19 15:47:42,638 p=334 u=akokov |  fatal: [test]: FAILED! => {"changed": false, "msg": "Failed to download metadata for repo 'docker-ce'", "rc": 1, "results": []}
2020-04-19 15:47:42,638 p=334 u=akokov |  PLAY RECAP **************************************************************************************************
2020-04-19 15:47:42,639 p=334 u=akokov |  test                       : ok=5    changed=1    unreachable=0    failed=1    skipped=0    rescued=0    ignored=0   
2020-04-19 16:03:39,087 p=3888 u=akokov |  ansible-playbook 2.8.3
  config file = /home/ansible/ansible.cfg
  configured module search path = ['/home/akokov/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/lib/python3/dist-packages/ansible
  executable location = /usr/bin/ansible-playbook
  python version = 3.7.5 (default, Nov 20 2019, 09:21:52) [GCC 9.2.1 20191008]
2020-04-19 16:03:39,087 p=3888 u=akokov |  Using /home/ansible/ansible.cfg as config file
2020-04-19 16:03:39,106 p=3888 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/docker.py) as it
seems to be invalid: No module named 'distutils.spawn'

2020-04-19 16:03:39,139 p=3888 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/iocage.py) as it
seems to be invalid: No module named 'distutils.spawn'

2020-04-19 16:03:39,139 p=3888 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/jail.py) as it seems
to be invalid: No module named 'distutils.spawn'

2020-04-19 16:03:39,139 p=3888 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/kubectl.py) as it
seems to be invalid: No module named 'distutils.spawn'

2020-04-19 16:03:39,140 p=3888 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/libvirt_lxc.py) as it
seems to be invalid: No module named 'distutils.spawn'

2020-04-19 16:03:39,141 p=3888 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/lxd.py) as it seems
to be invalid: No module named 'distutils.spawn'

2020-04-19 16:03:39,147 p=3888 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/oc.py) as it seems to
be invalid: No module named 'distutils.spawn'

2020-04-19 16:03:39,170 p=3888 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/zone.py) as it seems
to be invalid: No module named 'distutils.spawn'

2020-04-19 16:03:39,282 p=3888 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/callback/osx_say.py) as it seems
to be invalid: No module named 'distutils.spawn'

2020-04-19 16:03:39,283 p=3888 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/callback/say.py) as it seems to
be invalid: No module named 'distutils.spawn'

2020-04-19 16:03:39,290 p=3888 u=akokov |  PLAYBOOK: playbook-install.yml ******************************************************************************
2020-04-19 16:03:39,290 p=3888 u=akokov |  1 plays in playbook-install.yml
2020-04-19 16:03:39,317 p=3888 u=akokov |  PLAY [test] *************************************************************************************************
2020-04-19 16:03:39,324 p=3888 u=akokov |  TASK [Gathering Facts] **************************************************************************************
2020-04-19 16:03:39,324 p=3888 u=akokov |  task path: /home/ansible/playbook-install.yml:3
2020-04-19 16:03:40,362 p=3888 u=akokov |  ok: [test]
2020-04-19 16:03:40,362 p=3888 u=akokov |  META: ran handlers
2020-04-19 16:03:40,368 p=3888 u=akokov |  TASK [install latest Yum-Utils] *****************************************************************************
2020-04-19 16:03:40,368 p=3888 u=akokov |  task path: /home/ansible/playbook-install.yml:11
2020-04-19 16:03:41,494 p=3888 u=akokov |  fatal: [test]: FAILED! => {"changed": false, "msg": "Failed to download metadata for repo 'docker-ce'", "rc": 1, "results": []}
2020-04-19 16:03:41,495 p=3888 u=akokov |  PLAY RECAP **************************************************************************************************
2020-04-19 16:03:41,495 p=3888 u=akokov |  test                       : ok=1    changed=0    unreachable=0    failed=1    skipped=0    rescued=0    ignored=0   
2020-04-19 16:36:29,184 p=9359 u=akokov |  ansible-playbook 2.8.3
  config file = /home/ansible/ansible.cfg
  configured module search path = ['/home/akokov/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/lib/python3/dist-packages/ansible
  executable location = /usr/bin/ansible-playbook
  python version = 3.7.5 (default, Nov 20 2019, 09:21:52) [GCC 9.2.1 20191008]
2020-04-19 16:36:29,184 p=9359 u=akokov |  Using /home/ansible/ansible.cfg as config file
2020-04-19 16:36:29,203 p=9359 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/docker.py) as it seems to be invalid: No module named 'distutils.spawn'

2020-04-19 16:36:29,235 p=9359 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/iocage.py) as it seems to be invalid: No module named 'distutils.spawn'

2020-04-19 16:36:29,235 p=9359 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/jail.py) as it seems to be invalid: No module named 'distutils.spawn'

2020-04-19 16:36:29,236 p=9359 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/kubectl.py) as it seems to be invalid: No module named 'distutils.spawn'

2020-04-19 16:36:29,236 p=9359 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/libvirt_lxc.py) as it seems to be invalid: No module named 'distutils.spawn'

2020-04-19 16:36:29,238 p=9359 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/lxd.py) as it seems to be invalid: No module named 'distutils.spawn'

2020-04-19 16:36:29,244 p=9359 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/oc.py) as it seems to be invalid: No module named 'distutils.spawn'

2020-04-19 16:36:29,266 p=9359 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/zone.py) as it seems to be invalid: No module named 'distutils.spawn'

2020-04-19 16:36:29,379 p=9359 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/callback/osx_say.py) as it seems to be invalid: No module named 'distutils.spawn'

2020-04-19 16:36:29,380 p=9359 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/callback/say.py) as it seems to be invalid: No module named 'distutils.spawn'

2020-04-19 16:36:29,388 p=9359 u=akokov |  PLAYBOOK: playbook.yml **********************************************************************************************************************************************************************************************
2020-04-19 16:36:29,388 p=9359 u=akokov |  1 plays in playbook.yml
2020-04-19 16:36:29,408 p=9359 u=akokov |  PLAY [Install docker] ***********************************************************************************************************************************************************************************************
2020-04-19 16:36:29,410 p=9359 u=akokov |  META: ran handlers
2020-04-19 16:36:29,414 p=9359 u=akokov |  TASK [Install yum utils] ********************************************************************************************************************************************************************************************
2020-04-19 16:36:29,414 p=9359 u=akokov |  task path: /home/ansible/playbook.yml:7
2020-04-19 16:36:31,299 p=9359 u=akokov |  fatal: [test]: FAILED! => {"ansible_facts": {"discovered_interpreter_python": "/usr/libexec/platform-python"}, "changed": false, "msg": "Failed to download metadata for repo 'docker-ce'", "rc": 1, "results": []}
2020-04-19 16:36:31,300 p=9359 u=akokov |  PLAY RECAP **********************************************************************************************************************************************************************************************************
2020-04-19 16:36:31,300 p=9359 u=akokov |  test                       : ok=0    changed=0    unreachable=0    failed=1    skipped=0    rescued=0    ignored=0   
2020-05-01 17:38:01,743 p=16445 u=akokov |  ERROR! the playbook: l could not be found
2020-05-01 17:38:03,300 p=16454 u=akokov |   [WARNING]: provided hosts list is empty, only localhost is available. Note that the implicit localhost
does not match 'all'

2020-05-01 17:38:03,312 p=16454 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/docker.py) as it
seems to be invalid: No module named 'distutils.spawn'

2020-05-01 17:38:03,363 p=16454 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/iocage.py) as it
seems to be invalid: No module named 'distutils.spawn'

2020-05-01 17:38:03,363 p=16454 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/jail.py) as it
seems to be invalid: No module named 'distutils.spawn'

2020-05-01 17:38:03,363 p=16454 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/kubectl.py) as it
seems to be invalid: No module named 'distutils.spawn'

2020-05-01 17:38:03,364 p=16454 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/libvirt_lxc.py) as
it seems to be invalid: No module named 'distutils.spawn'

2020-05-01 17:38:03,367 p=16454 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/lxd.py) as it
seems to be invalid: No module named 'distutils.spawn'

2020-05-01 17:38:03,375 p=16454 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/oc.py) as it seems
to be invalid: No module named 'distutils.spawn'

2020-05-01 17:38:03,418 p=16454 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/zone.py) as it
seems to be invalid: No module named 'distutils.spawn'

2020-05-01 17:38:03,651 p=16454 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/callback/osx_say.py) as it
seems to be invalid: No module named 'distutils.spawn'

2020-05-01 17:38:03,652 p=16454 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/callback/say.py) as it seems
to be invalid: No module named 'distutils.spawn'

2020-05-01 17:38:03,689 p=16454 u=akokov |   [WARNING]: Could not match supplied host pattern, ignoring: test

2020-05-01 17:38:03,690 p=16454 u=akokov |  PLAY [test] **********************************************************************************************
2020-05-01 17:38:03,690 p=16454 u=akokov |  skipping: no hosts matched
2020-05-01 17:38:03,690 p=16454 u=akokov |  PLAY RECAP ***********************************************************************************************
2020-05-01 17:38:26,730 p=16518 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/docker.py) as it
seems to be invalid: No module named 'distutils.spawn'

2020-05-01 17:38:26,762 p=16518 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/iocage.py) as it
seems to be invalid: No module named 'distutils.spawn'

2020-05-01 17:38:26,762 p=16518 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/jail.py) as it
seems to be invalid: No module named 'distutils.spawn'

2020-05-01 17:38:26,763 p=16518 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/kubectl.py) as it
seems to be invalid: No module named 'distutils.spawn'

2020-05-01 17:38:26,763 p=16518 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/libvirt_lxc.py) as
it seems to be invalid: No module named 'distutils.spawn'

2020-05-01 17:38:26,764 p=16518 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/lxd.py) as it
seems to be invalid: No module named 'distutils.spawn'

2020-05-01 17:38:26,771 p=16518 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/oc.py) as it seems
to be invalid: No module named 'distutils.spawn'

2020-05-01 17:38:26,793 p=16518 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/connection/zone.py) as it
seems to be invalid: No module named 'distutils.spawn'

2020-05-01 17:38:26,904 p=16518 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/callback/osx_say.py) as it
seems to be invalid: No module named 'distutils.spawn'

2020-05-01 17:38:26,906 p=16518 u=akokov |   [WARNING]: Skipping plugin (/usr/lib/python3/dist-packages/ansible/plugins/callback/say.py) as it seems
to be invalid: No module named 'distutils.spawn'

2020-05-01 17:38:26,933 p=16518 u=akokov |  PLAY [test] **********************************************************************************************
2020-05-01 17:38:26,939 p=16518 u=akokov |  TASK [Gathering Facts] ***********************************************************************************
2020-05-01 17:38:28,102 p=16518 u=akokov |  ok: [test]
2020-05-01 17:38:28,110 p=16518 u=akokov |  TASK [install latest Yum-Utils] **************************************************************************
2020-05-01 17:38:34,068 p=16518 u=akokov |  fatal: [test]: FAILED! => {"changed": false, "msg": "Failed to download metadata for repo 'docker-ce'", "rc": 1, "results": []}
2020-05-01 17:38:34,069 p=16518 u=akokov |  PLAY RECAP ***********************************************************************************************
2020-05-01 17:38:34,069 p=16518 u=akokov |  test                       : ok=1    changed=0    unreachable=0    failed=1    skipped=0    rescued=0    ignored=0   
